stdlib/dbm/__init__.pyi

```diff
@@ -23,4 +23,4 @@ class _error(Exception): ...
 error = Tuple[Type[_error], Type[OSError]]
 
 def whichdb(filename: str) -> str: ...
-def open(file: str, flag: Literal["r", "w", "c", "n"] = ..., mode: int = ...) -> _Database: ...
+def open(file: str, flag: Literal["r", "w", "c", "n", "f", "s", "u"] = ..., mode: int = ...) -> _Database: ...
```

Comment:
The issue asked for `"cs"`, which this still wouldn't allow. The documentation says there is an `open_flags` variable that will give all possibilities.

Reply:
Alright!

Reply:
can't find that variable [here](https://github.com/python/cpython/blob/3.9/Lib/dbm/__init__.py). The alternative way is to write all the combinations?

Reply:
It's here: https://github.com/python/cpython/blob/main/Modules/_gdbmmodule.c#L679. But looks like it isn't quite right either; it gives all individual characters but not all flag combinations. Looks like there are 43 possible combinations.

Reply:
So I need to build that variable of all the combinations. Sounds easy.

Reply:
lemme do this.

Reply:
Oh that's not true, the first character must be one of `rwcn`. The other three flags can follow the first character in any order. I think that makes `4 * (1 + 3 + 3 * 2 + 3 * 2 * 1) = 64` possible strings; the 43 in my previous comment was wrong.

Reply:
How about this approach:

```python
>>> from itertools import permutations 
>>> open_flags =  ["".join(tup) for i in range(1, 7) for tup in permutations(["r", "w", "c", "n", "f", "s", "u"], i)]
>>> open_flags
['r', 'w', 'c', 'n', 'f', 's', 'u', 'rw', 'rc', 'rn', 'rf', 'rs', 'ru', 'wr', 'wc', 'wn', 'wf', 'ws', 'wu', 'cr', 'cw', 'cn', 'cf', 'cs', 'cu', 'nr', 'nw', 'nc', 'nf', 'ns', 'nu', 'fr', 'fw', 'fc', 'fn', 'fs', 'fu', 'sr', 'sw', 'sc', 'sn', 'sf', 'su', 'ur', 'uw', 'uc', 'un', 'uf', 'us', 'rwc', 'rwn', 'rwf', 'rws', 'rwu', 'rcw', 'rcn', 'rcf', 'rcs', 'rcu', 'rnw', 'rnc', 'rnf', 'rns', 'rnu', 'rfw', 'rfc', 'rfn', 'rfs', 'rfu', 'rsw', 'rsc', 'rsn', 'rsf', 'rsu', 'ruw', 'ruc', 'run', 'ruf', 'rus', 'wrc', 'wrn', 'wrf', 'wrs', 'wru', 'wcr', 'wcn', 'wcf', 'wcs', 'wcu', 'wnr', 'wnc', 'wnf', 'wns', 'wnu', 'wfr', 'wfc', 'wfn', 'wfs', 'wfu', 'wsr', 'wsc', 'wsn', 'wsf', 'wsu', 'wur', 'wuc', 'wun', 'wuf', 'wus', 'crw', 'crn', 'crf', 'crs', 'cru', 'cwr', 'cwn', 'cwf', 'cws', 'cwu', 'cnr', 'cnw', 'cnf', 'cns', 'cnu', 'cfr', 'cfw', 'cfn', 'cfs', 'cfu', 'csr', 'csw', 'csn', 'csf', 'csu', 'cur', 'cuw', 'cun', 'cuf', 'cus', 'nrw', 'nrc', 'nrf', 'nrs', 'nru', 'nwr', 'nwc', 'nwf', 'nws', 'nwu', 'ncr', 'ncw', 'ncf', 'ncs', 'ncu', 'nfr', 'nfw', 'nfc', 'nfs', 'nfu', 'nsr', 'nsw', 'nsc', 'nsf', 'nsu', 'nur', 'nuw', 'nuc', 'nuf', 'nus', 'frw', 'frc', 'frn', 'frs', 'fru', 'fwr', 'fwc', 'fwn', 'fws', 'fwu', 'fcr', 'fcw', 'fcn', 'fcs', 'fcu', 'fnr', 'fnw', 'fnc', 'fns', 'fnu', 'fsr', 'fsw', 'fsc', 'fsn', 'fsu', 'fur', 'fuw', 'fuc', 'fun', 'fus', 'srw', 'src', 'srn', 'srf', 'sru', 'swr', 'swc', 'swn', 'swf', 'swu', 'scr', 'scw', 'scn', 'scf', 'scu', 'snr', 'snw', 'snc', 'snf', 'snu', 'sfr', 'sfw', 'sfc', 'sfn', 'sfu', 'sur', 'suw', 'suc', 'sun', 'suf', 'urw', 'urc', 'urn', 'urf', 'urs', 'uwr', 'uwc', 'uwn', 'uwf', 'uws', 'ucr', 'ucw', 'ucn', 'ucf', 'ucs', 'unr', 'unw', 'unc', 'unf', 'uns', 'ufr', 'ufw', 'ufc', 'ufn', 'ufs', 'usr', 'usw', 'usc', 'usn', 'usf', 'rwcn', 'rwcf', 'rwcs', 'rwcu', 'rwnc', 'rwnf', 'rwns', 'rwnu', 'rwfc', 'rwfn', 'rwfs', 'rwfu', 'rwsc', 'rwsn', 'rwsf', 'rwsu', 'rwuc', 'rwun', 'rwuf', 'rwus', 'rcwn', 'rcwf', 'rcws', 'rcwu', 'rcnw', 'rcnf', 'rcns', 'rcnu', 'rcfw', 'rcfn', 'rcfs', 'rcfu', 'rcsw', 'rcsn', 'rcsf', 'rcsu', 'rcuw', 'rcun', 'rcuf', 'rcus', 'rnwc', 'rnwf', 'rnws', 'rnwu', 'rncw', 'rncf', 'rncs', 'rncu', 'rnfw', 'rnfc', 'rnfs', 'rnfu', 'rnsw', 'rnsc', 'rnsf', 'rnsu', 'rnuw', 'rnuc', 'rnuf', 'rnus', 'rfwc', 'rfwn', 'rfws', 'rfwu', 'rfcw', 'rfcn', 'rfcs', 'rfcu', 'rfnw', 'rfnc', 'rfns', 'rfnu', 'rfsw', 'rfsc', 'rfsn', 'rfsu', 'rfuw', 'rfuc', 'rfun', 'rfus', 'rswc', 'rswn', 'rswf', 'rswu', 'rscw', 'rscn', 'rscf', 'rscu', 'rsnw', 'rsnc', 'rsnf', 'rsnu', 'rsfw', 'rsfc', 'rsfn', 'rsfu', 'rsuw', 'rsuc', 'rsun', 'rsuf', 'ruwc', 'ruwn', 'ruwf', 'ruws', 'rucw', 'rucn', 'rucf', 'rucs', 'runw', 'runc', 'runf', 'runs', 'rufw', 'rufc', 'rufn', 'rufs', 'rusw', 'rusc', 'rusn', 'rusf', 'wrcn', 'wrcf', 'wrcs', 'wrcu', 'wrnc', 'wrnf', 'wrns', 'wrnu', 'wrfc', 'wrfn', 'wrfs', 'wrfu', 'wrsc', 'wrsn', 'wrsf', 'wrsu', 'wruc', 'wrun', 'wruf', 'wrus', 'wcrn', 'wcrf', 'wcrs', 'wcru', 'wcnr', 'wcnf', 'wcns', 'wcnu', 'wcfr', 'wcfn', 'wcfs', 'wcfu', 'wcsr', 'wcsn', 'wcsf', 'wcsu', 'wcur', 'wcun', 'wcuf', 'wcus', 'wnrc', 'wnrf', 'wnrs', 'wnru', 'wncr', 'wncf', 'wncs', 'wncu', 'wnfr', 'wnfc', 'wnfs', 'wnfu', 'wnsr', 'wnsc', 'wnsf', 'wnsu', 'wnur', 'wnuc', 'wnuf', 'wnus', 'wfrc', 'wfrn', 'wfrs', 'wfru', 'wfcr', 'wfcn', 'wfcs', 'wfcu', 'wfnr', 'wfnc', 'wfns', 'wfnu', 'wfsr', 'wfsc', 'wfsn', 'wfsu', 'wfur', 'wfuc', 'wfun', 'wfus', 'wsrc', 'wsrn', 'wsrf', 'wsru', 'wscr', 'wscn', 'wscf', 'wscu', 'wsnr', 'wsnc', 'wsnf', 'wsnu', 'wsfr', 'wsfc', 'wsfn', 'wsfu', 'wsur', 'wsuc', 'wsun', 'wsuf', 'wurc', 'wurn', 'wurf', 'wurs', 'wucr', 'wucn', 'wucf', 'wucs', 'wunr', 'wunc', 'wunf', 'wuns', 'wufr', 'wufc', 'wufn', 'wufs', 'wusr', 'wusc', 'wusn', 'wusf', 'crwn', 'crwf', 'crws', 'crwu', 'crnw', 'crnf', 'crns', 'crnu', 'crfw', 'crfn', 'crfs', 'crfu', 'crsw', 'crsn', 'crsf', 'crsu', 'cruw', 'crun', 'cruf', 'crus', 'cwrn', 'cwrf', 'cwrs', 'cwru', 'cwnr', 'cwnf', 'cwns', 'cwnu', 'cwfr', 'cwfn', 'cwfs', 'cwfu', 'cwsr', 'cwsn', 'cwsf', 'cwsu', 'cwur', 'cwun', 'cwuf', 'cwus', 'cnrw', 'cnrf', 'cnrs', 'cnru', 'cnwr', 'cnwf', 'cnws', 'cnwu', 'cnfr', 'cnfw', 'cnfs', 'cnfu', 'cnsr', 'cnsw', 'cnsf', 'cnsu', 'cnur', 'cnuw', 'cnuf', 'cnus', 'cfrw', 'cfrn', 'cfrs', 'cfru', 'cfwr', 'cfwn', 'cfws', 'cfwu', 'cfnr', 'cfnw', 'cfns', 'cfnu', 'cfsr', 'cfsw', 'cfsn', 'cfsu', 'cfur', 'cfuw', 'cfun', 'cfus', 'csrw', 'csrn', 'csrf', 'csru', 'cswr', 'cswn', 'cswf', 'cswu', 'csnr', 'csnw', 'csnf', 'csnu', 'csfr', 'csfw', 'csfn', 'csfu', 'csur', 'csuw', 'csun', 'csuf', 'curw', 'curn', 'curf', 'curs', 'cuwr', 'cuwn', 'cuwf', 'cuws', 'cunr', 'cunw', 'cunf', 'cuns', 'cufr', 'cufw', 'cufn', 'cufs', 'cusr', 'cusw', 'cusn', 'cusf', 'nrwc', 'nrwf', 'nrws', 'nrwu', 'nrcw', 'nrcf', 'nrcs', 'nrcu', 'nrfw', 'nrfc', 'nrfs', 'nrfu', 'nrsw', 'nrsc', 'nrsf', 'nrsu', 'nruw', 'nruc', 'nruf', 'nrus', 'nwrc', 'nwrf', 'nwrs', 'nwru', 'nwcr', 'nwcf', 'nwcs', 'nwcu', 'nwfr', 'nwfc', 'nwfs', 'nwfu', 'nwsr', 'nwsc', 'nwsf', 'nwsu', 'nwur', 'nwuc', 'nwuf', 'nwus', 'ncrw', 'ncrf', 'ncrs', 'ncru', 'ncwr', 'ncwf', 'ncws', 'ncwu', 'ncfr', 'ncfw', 'ncfs', 'ncfu', 'ncsr', 'ncsw', 'ncsf', 'ncsu', 'ncur', 'ncuw', 'ncuf', 'ncus', 'nfrw', 'nfrc', 'nfrs', 'nfru', 'nfwr', 'nfwc', 'nfws', 'nfwu', 'nfcr', 'nfcw', 'nfcs', 'nfcu', 'nfsr', 'nfsw', 'nfsc', 'nfsu', 'nfur', 'nfuw', 'nfuc', 'nfus', 'nsrw', 'nsrc', 'nsrf', 'nsru', 'nswr', 'nswc', 'nswf', 'nswu', 'nscr', 'nscw', 'nscf', 'nscu', 'nsfr', 'nsfw', 'nsfc', 'nsfu', 'nsur', 'nsuw', 'nsuc', 'nsuf', 'nurw', 'nurc', 'nurf', 'nurs', 'nuwr', 'nuwc', 'nuwf', 'nuws', 'nucr', 'nucw', 'nucf', 'nucs', 'nufr', 'nufw', 'nufc', 'nufs', 'nusr', 'nusw', 'nusc', 'nusf', 'frwc', 'frwn', 'frws', 'frwu', 'frcw', 'frcn', 'frcs', 'frcu', 'frnw', 'frnc', 'frns', 'frnu', 'frsw', 'frsc', 'frsn', 'frsu', 'fruw', 'fruc', 'frun', 'frus', 'fwrc', 'fwrn', 'fwrs', 'fwru', 'fwcr', 'fwcn', 'fwcs', 'fwcu', 'fwnr', 'fwnc', 'fwns', 'fwnu', 'fwsr', 'fwsc', 'fwsn', 'fwsu', 'fwur', 'fwuc', 'fwun', 'fwus', 'fcrw', 'fcrn', 'fcrs', 'fcru', 'fcwr', 'fcwn', 'fcws', 'fcwu', 'fcnr', 'fcnw', 'fcns', 'fcnu', 'fcsr', 'fcsw', 'fcsn', 'fcsu', 'fcur', 'fcuw', 'fcun', 'fcus', 'fnrw', 'fnrc', 'fnrs', 'fnru', 'fnwr', 'fnwc', 'fnws', 'fnwu', 'fncr', 'fncw', 'fncs', 'fncu', 'fnsr', 'fnsw', 'fnsc', 'fnsu', 'fnur', 'fnuw', 'fnuc', 'fnus', 'fsrw', 'fsrc', 'fsrn', 'fsru', 'fswr', 'fswc', 'fswn', 'fswu', 'fscr', 'fscw', 'fscn', 'fscu', 'fsnr', 'fsnw', 'fsnc', 'fsnu', 'fsur', 'fsuw', 'fsuc', 'fsun', 'furw', 'furc', 'furn', 'furs', 'fuwr', 'fuwc', 'fuwn', 'fuws', 'fucr', 'fucw', 'fucn', 'fucs', 'funr', 'funw', 'func', 'funs', 'fusr', 'fusw', 'fusc', 'fusn', 'srwc', 'srwn', 'srwf', 'srwu', 'srcw', 'srcn', 'srcf', 'srcu', 'srnw', 'srnc', 'srnf', 'srnu', 'srfw', 'srfc', 'srfn', 'srfu', 'sruw', 'sruc', 'srun', 'sruf', 'swrc', 'swrn', 'swrf', 'swru', 'swcr', 'swcn', 'swcf', 'swcu', 'swnr', 'swnc', 'swnf', 'swnu', 'swfr', 'swfc', 'swfn', 'swfu', 'swur', 'swuc', 'swun', 'swuf', 'scrw', 'scrn', 'scrf', 'scru', 'scwr', 'scwn', 'scwf', 'scwu', 'scnr', 'scnw', 'scnf', 'scnu', 'scfr', 'scfw', 'scfn', 'scfu', 'scur', 'scuw', 'scun', 'scuf', 'snrw', 'snrc', 'snrf', 'snru', 'snwr', 'snwc', 'snwf', 'snwu', 'sncr', 'sncw', 'sncf', 'sncu', 'snfr', 'snfw', 'snfc', 'snfu', 'snur', 'snuw', 'snuc', 'snuf', 'sfrw', 'sfrc', 'sfrn', 'sfru', 'sfwr', 'sfwc', 'sfwn', 'sfwu', 'sfcr', 'sfcw', 'sfcn', 'sfcu', 'sfnr', 'sfnw', 'sfnc', 'sfnu', 'sfur', 'sfuw', 'sfuc', 'sfun', 'surw', 'surc', 'surn', 'surf', 'suwr', 'suwc', 'suwn', 'suwf', 'sucr', 'sucw', 'sucn', 'sucf', 'sunr', 'sunw', 'sunc', 'sunf', 'sufr', 'sufw', 'sufc', 'sufn', 'urwc', 'urwn', 'urwf', 'urws', 'urcw', 'urcn', 'urcf', 'urcs', 'urnw', 'urnc', 'urnf', 'urns', 'urfw', 'urfc', 'urfn', 'urfs', 'ursw', 'ursc', 'ursn', 'ursf', 'uwrc', 'uwrn', 'uwrf', 'uwrs', 'uwcr', 'uwcn', 'uwcf', 'uwcs', 'uwnr', 'uwnc', 'uwnf', 'uwns', 'uwfr', 'uwfc', 'uwfn', 'uwfs', 'uwsr', 'uwsc', 'uwsn', 'uwsf', 'ucrw', 'ucrn', 'ucrf', 'ucrs', 'ucwr', 'ucwn', 'ucwf', 'ucws', 'ucnr', 'ucnw', 'ucnf', 'ucns', 'ucfr', 'ucfw', 'ucfn', 'ucfs', 'ucsr', 'ucsw', 'ucsn', 'ucsf', 'unrw', 'unrc', 'unrf', 'unrs', 'unwr', 'unwc', 'unwf', 'unws', 'uncr', 'uncw', 'uncf', 'uncs', 'unfr', 'unfw', 'unfc', 'unfs', 'unsr', 'unsw', 'unsc', 'unsf', 'ufrw', 'ufrc', 'ufrn', 'ufrs', 'ufwr', 'ufwc', 'ufwn', 'ufws', 'ufcr', 'ufcw', 'ufcn', 'ufcs', 'ufnr', 'ufnw', 'ufnc', 'ufns', 'ufsr', 'ufsw', 'ufsc', 'ufsn', 'usrw', 'usrc', 'usrn', 'usrf', 'uswr', 'uswc', 'uswn', 'uswf', 'uscr', 'uscw', 'uscn', 'uscf', 'usnr', 'usnw', 'usnc', 'usnf', 'usfr', 'usfw', 'usfc', 'usfn', 'rwcnf', 'rwcns', 'rwcnu', 'rwcfn', 'rwcfs', 'rwcfu', 'rwcsn', 'rwcsf', 'rwcsu', 'rwcun', 'rwcuf', 'rwcus', 'rwncf', 'rwncs', 'rwncu', 'rwnfc', 'rwnfs', 'rwnfu', 'rwnsc', 'rwnsf', 'rwnsu', 'rwnuc', 'rwnuf', 'rwnus', 'rwfcn', 'rwfcs', 'rwfcu', 'rwfnc', 'rwfns', 'rwfnu', 'rwfsc', 'rwfsn', 'rwfsu', 'rwfuc', 'rwfun', 'rwfus', 'rwscn', 'rwscf', 'rwscu', 'rwsnc', 'rwsnf', 'rwsnu', 'rwsfc', 'rwsfn', 'rwsfu', 'rwsuc', 'rwsun', 'rwsuf', 'rwucn', 'rwucf', 'rwucs', 'rwunc', 'rwunf', 'rwuns', 'rwufc', 'rwufn', 'rwufs', 'rwusc', 'rwusn', 'rwusf', 'rcwnf', 'rcwns', 'rcwnu', 'rcwfn', 'rcwfs', 'rcwfu', 'rcwsn', 'rcwsf', 'rcwsu', 'rcwun', 'rcwuf', 'rcwus', 'rcnwf', 'rcnws', 'rcnwu', 'rcnfw', 'rcnfs', 'rcnfu', 'rcnsw', 'rcnsf', 'rcnsu', 'rcnuw', 'rcnuf', 'rcnus', 'rcfwn', 'rcfws', 'rcfwu', 'rcfnw', 'rcfns', 'rcfnu', 'rcfsw', 'rcfsn', 'rcfsu', 'rcfuw', 'rcfun', 'rcfus', 'rcswn', 'rcswf', 'rcswu', 'rcsnw', 'rcsnf', 'rcsnu', 'rcsfw', 'rcsfn', 'rcsfu', 'rcsuw', 'rcsun', 'rcsuf', 'rcuwn', 'rcuwf', 'rcuws', 'rcunw', 'rcunf', 'rcuns', 'rcufw', 'rcufn', 'rcufs', 'rcusw', 'rcusn', 'rcusf', 'rnwcf', 'rnwcs', 'rnwcu', 'rnwfc', 'rnwfs', 'rnwfu', 'rnwsc', 'rnwsf', 'rnwsu', 'rnwuc', 'rnwuf', 'rnwus', 'rncwf', 'rncws', 'rncwu', 'rncfw', 'rncfs', 'rncfu', 'rncsw', 'rncsf', 'rncsu', 'rncuw', 'rncuf', 'rncus', 'rnfwc', 'rnfws', 'rnfwu', 'rnfcw', 'rnfcs', 'rnfcu', 'rnfsw', 'rnfsc', 'rnfsu', 'rnfuw', 'rnfuc', 'rnfus', 'rnswc', 'rnswf', 'rnswu', 'rnscw', 'rnscf', 'rnscu', 'rnsfw', 'rnsfc', 'rnsfu', 'rnsuw', 'rnsuc', 'rnsuf', 'rnuwc', 'rnuwf', 'rnuws', 'rnucw', 'rnucf', 'rnucs', 'rnufw', 'rnufc', 'rnufs', 'rnusw', 'rnusc', 'rnusf', 'rfwcn', 'rfwcs', 'rfwcu', 'rfwnc', 'rfwns', 'rfwnu', 'rfwsc', 'rfwsn', 'rfwsu', 'rfwuc', 'rfwun', 'rfwus', 'rfcwn', 'rfcws', 'rfcwu', 'rfcnw', 'rfcns', 'rfcnu', 'rfcsw', 'rfcsn', 'rfcsu', 'rfcuw', 'rfcun', 'rfcus', 'rfnwc', 'rfnws', 'rfnwu', 'rfncw', 'rfncs', 'rfncu', 'rfnsw', 'rfnsc', 'rfnsu', 'rfnuw', 'rfnuc', 'rfnus', 'rfswc', 'rfswn', 'rfswu', 'rfscw', 'rfscn', 'rfscu', 'rfsnw', 'rfsnc', 'rfsnu', 'rfsuw', 'rfsuc', 'rfsun', 'rfuwc', 'rfuwn', 'rfuws', 'rfucw', 'rfucn', 'rfucs', 'rfunw', 'rfunc', 'rfuns', 'rfusw', 'rfusc', 'rfusn', 'rswcn', 'rswcf', 'rswcu', 'rswnc', 'rswnf', 'rswnu', 'rswfc', 'rswfn', 'rswfu', 'rswuc', 'rswun', 'rswuf', 'rscwn', 'rscwf', 'rscwu', 'rscnw', 'rscnf', 'rscnu', 'rscfw', 'rscfn', 'rscfu', 'rscuw', 'rscun', 'rscuf', 'rsnwc', 'rsnwf', 'rsnwu', 'rsncw', 'rsncf', 'rsncu', 'rsnfw', 'rsnfc', 'rsnfu', 'rsnuw', 'rsnuc', 'rsnuf', 'rsfwc', 'rsfwn', 'rsfwu', 'rsfcw', 'rsfcn', 'rsfcu', 'rsfnw', 'rsfnc', 'rsfnu', 'rsfuw', 'rsfuc', 'rsfun', 'rsuwc', 'rsuwn', 'rsuwf', 'rsucw', 'rsucn', 'rsucf', 'rsunw', 'rsunc', 'rsunf', 'rsufw', 'rsufc', 'rsufn', 'ruwcn', 'ruwcf', 'ruwcs', 'ruwnc', 'ruwnf', 'ruwns', 'ruwfc', 'ruwfn', 'ruwfs', 'ruwsc', 'ruwsn', 'ruwsf', 'rucwn', 'rucwf', 'rucws', 'rucnw', 'rucnf', 'rucns', 'rucfw', 'rucfn', 'rucfs', 'rucsw', 'rucsn', 'rucsf', 'runwc', 'runwf', 'runws', 'runcw', 'runcf', 'runcs', 'runfw', 'runfc', 'runfs', 'runsw', 'runsc', 'runsf', 'rufwc', 'rufwn', 'rufws', 'rufcw', 'rufcn', 'rufcs', 'rufnw', 'rufnc', 'rufns', 'rufsw', 'rufsc', 'rufsn', 'ruswc', 'ruswn', 'ruswf', 'ruscw', 'ruscn', 'ruscf', 'rusnw', 'rusnc', 'rusnf', 'rusfw', 'rusfc', 'rusfn', 'wrcnf', 'wrcns', 'wrcnu', 'wrcfn', 'wrcfs', 'wrcfu', 'wrcsn', 'wrcsf', 'wrcsu', 'wrcun', 'wrcuf', 'wrcus', 'wrncf', 'wrncs', 'wrncu', 'wrnfc', 'wrnfs', 'wrnfu', 'wrnsc', 'wrnsf', 'wrnsu', 'wrnuc', 'wrnuf', 'wrnus', 'wrfcn', 'wrfcs', 'wrfcu', 'wrfnc', 'wrfns', 'wrfnu', 'wrfsc', 'wrfsn', 'wrfsu', 'wrfuc', 'wrfun', 'wrfus', 'wrscn', 'wrscf', 'wrscu', 'wrsnc', 'wrsnf', 'wrsnu', 'wrsfc', 'wrsfn', 'wrsfu', 'wrsuc', 'wrsun', 'wrsuf', 'wrucn', 'wrucf', 'wrucs', 'wrunc', 'wrunf', 'wruns', 'wrufc', 'wrufn', 'wrufs', 'wrusc', 'wrusn', 'wrusf', 'wcrnf', 'wcrns', 'wcrnu', 'wcrfn', 'wcrfs', 'wcrfu', 'wcrsn', 'wcrsf', 'wcrsu', 'wcrun', 'wcruf', 'wcrus', 'wcnrf', 'wcnrs', 'wcnru', 'wcnfr', 'wcnfs', 'wcnfu', 'wcnsr', 'wcnsf', 'wcnsu', 'wcnur', 'wcnuf', 'wcnus', 'wcfrn', 'wcfrs', 'wcfru', 'wcfnr', 'wcfns', 'wcfnu', 'wcfsr', 'wcfsn', 'wcfsu', 'wcfur', 'wcfun', 'wcfus', 'wcsrn', 'wcsrf', 'wcsru', 'wcsnr', 'wcsnf', 'wcsnu', 'wcsfr', 'wcsfn', 'wcsfu', 'wcsur', 'wcsun', 'wcsuf', 'wcurn', 'wcurf', 'wcurs', 'wcunr', 'wcunf', 'wcuns', 'wcufr', 'wcufn', 'wcufs', 'wcusr', 'wcusn', 'wcusf', 'wnrcf', 'wnrcs', 'wnrcu', 'wnrfc', 'wnrfs', 'wnrfu', 'wnrsc', 'wnrsf', 'wnrsu', 'wnruc', 'wnruf', 'wnrus', 'wncrf', 'wncrs', 'wncru', 'wncfr', 'wncfs', 'wncfu', 'wncsr', 'wncsf', 'wncsu', 'wncur', 'wncuf', 'wncus', 'wnfrc', 'wnfrs', 'wnfru', 'wnfcr', 'wnfcs', 'wnfcu', 'wnfsr', 'wnfsc', 'wnfsu', 'wnfur', 'wnfuc', 'wnfus', 'wnsrc', 'wnsrf', 'wnsru', 'wnscr', 'wnscf', 'wnscu', 'wnsfr', 'wnsfc', 'wnsfu', 'wnsur', 'wnsuc', 'wnsuf', 'wnurc', 'wnurf', 'wnurs', 'wnucr', 'wnucf', 'wnucs', 'wnufr', 'wnufc', 'wnufs', 'wnusr', 'wnusc', 'wnusf', 'wfrcn', 'wfrcs', 'wfrcu', 'wfrnc', 'wfrns', 'wfrnu', 'wfrsc', 'wfrsn', 'wfrsu', 'wfruc', 'wfrun', 'wfrus', 'wfcrn', 'wfcrs', 'wfcru', 'wfcnr', 'wfcns', 'wfcnu', 'wfcsr', 'wfcsn', 'wfcsu', 'wfcur', 'wfcun', 'wfcus', 'wfnrc', 'wfnrs', 'wfnru', 'wfncr', 'wfncs', 'wfncu', 'wfnsr', 'wfnsc', 'wfnsu', 'wfnur', 'wfnuc', 'wfnus', 'wfsrc', 'wfsrn', 'wfsru', 'wfscr', 'wfscn', 'wfscu', 'wfsnr', 'wfsnc', 'wfsnu', 'wfsur', 'wfsuc', 'wfsun', 'wfurc', 'wfurn', 'wfurs', 'wfucr', 'wfucn', 'wfucs', 'wfunr', 'wfunc', 'wfuns', 'wfusr', 'wfusc', 'wfusn', 'wsrcn', 'wsrcf', 'wsrcu', 'wsrnc', 'wsrnf', 'wsrnu', 'wsrfc', 'wsrfn', 'wsrfu', 'wsruc', 'wsrun', 'wsruf', 'wscrn', 'wscrf', 'wscru', 'wscnr', 'wscnf', 'wscnu', 'wscfr', 'wscfn', 'wscfu', 'wscur', 'wscun', 'wscuf', 'wsnrc', 'wsnrf', 'wsnru', 'wsncr', 'wsncf', 'wsncu', 'wsnfr', 'wsnfc', 'wsnfu', 'wsnur', 'wsnuc', 'wsnuf', 'wsfrc', 'wsfrn', 'wsfru', 'wsfcr', 'wsfcn', 'wsfcu', 'wsfnr', 'wsfnc', 'wsfnu', 'wsfur', 'wsfuc', 'wsfun', 'wsurc', 'wsurn', 'wsurf', 'wsucr', 'wsucn', 'wsucf', 'wsunr', 'wsunc', 'wsunf', 'wsufr', 'wsufc', 'wsufn', 'wurcn', 'wurcf', 'wurcs', 'wurnc', 'wurnf', 'wurns', 'wurfc', 'wurfn', 'wurfs', 'wursc', 'wursn', 'wursf', 'wucrn', 'wucrf', 'wucrs', 'wucnr', 'wucnf', 'wucns', 'wucfr', 'wucfn', 'wucfs', 'wucsr', 'wucsn', 'wucsf', 'wunrc', 'wunrf', 'wunrs', 'wuncr', 'wuncf', 'wuncs', 'wunfr', 'wunfc', 'wunfs', 'wunsr', 'wunsc', 'wunsf', 'wufrc', 'wufrn', 'wufrs', 'wufcr', 'wufcn', 'wufcs', 'wufnr', 'wufnc', 'wufns', 'wufsr', 'wufsc', 'wufsn', 'wusrc', 'wusrn', 'wusrf', 'wuscr', 'wuscn', 'wuscf', 'wusnr', 'wusnc', 'wusnf', 'wusfr', 'wusfc', 'wusfn', 'crwnf', 'crwns', 'crwnu', 'crwfn', 'crwfs', 'crwfu', 'crwsn', 'crwsf', 'crwsu', 'crwun', 'crwuf', 'crwus', 'crnwf', 'crnws', 'crnwu', 'crnfw', 'crnfs', 'crnfu', 'crnsw', 'crnsf', 'crnsu', 'crnuw', 'crnuf', 'crnus', 'crfwn', 'crfws', 'crfwu', 'crfnw', 'crfns', 'crfnu', 'crfsw', 'crfsn', 'crfsu', 'crfuw', 'crfun', 'crfus', 'crswn', 'crswf', 'crswu', 'crsnw', 'crsnf', 'crsnu', 'crsfw', 'crsfn', 'crsfu', 'crsuw', 'crsun', 'crsuf', 'cruwn', 'cruwf', 'cruws', 'crunw', 'crunf', 'cruns', 'crufw', 'crufn', 'crufs', 'crusw', 'crusn', 'crusf', 'cwrnf', 'cwrns', 'cwrnu', 'cwrfn', 'cwrfs', 'cwrfu', 'cwrsn', 'cwrsf', 'cwrsu', 'cwrun', 'cwruf', 'cwrus', 'cwnrf', 'cwnrs', 'cwnru', 'cwnfr', 'cwnfs', 'cwnfu', 'cwnsr', 'cwnsf', 'cwnsu', 'cwnur', 'cwnuf', 'cwnus', 'cwfrn', 'cwfrs', 'cwfru', 'cwfnr', 'cwfns', 'cwfnu', 'cwfsr', 'cwfsn', 'cwfsu', 'cwfur', 'cwfun', 'cwfus', 'cwsrn', 'cwsrf', 'cwsru', 'cwsnr', 'cwsnf', 'cwsnu', 'cwsfr', 'cwsfn', 'cwsfu', 'cwsur', 'cwsun', 'cwsuf', 'cwurn', 'cwurf', 'cwurs', 'cwunr', 'cwunf', 'cwuns', 'cwufr', 'cwufn', 'cwufs', 'cwusr', 'cwusn', 'cwusf', 'cnrwf', 'cnrws', 'cnrwu', 'cnrfw', 'cnrfs', 'cnrfu', 'cnrsw', 'cnrsf', 'cnrsu', 'cnruw', 'cnruf', 'cnrus', 'cnwrf', 'cnwrs', 'cnwru', 'cnwfr', 'cnwfs', 'cnwfu', 'cnwsr', 'cnwsf', 'cnwsu', 'cnwur', 'cnwuf', 'cnwus', 'cnfrw', 'cnfrs', 'cnfru', 'cnfwr', 'cnfws', 'cnfwu', 'cnfsr', 'cnfsw', 'cnfsu', 'cnfur', 'cnfuw', 'cnfus', 'cnsrw', 'cnsrf', 'cnsru', 'cnswr', 'cnswf', 'cnswu', 'cnsfr', 'cnsfw', 'cnsfu', 'cnsur', 'cnsuw', 'cnsuf', 'cnurw', 'cnurf', 'cnurs', 'cnuwr', 'cnuwf', 'cnuws', 'cnufr', 'cnufw', 'cnufs', 'cnusr', 'cnusw', 'cnusf', 'cfrwn', 'cfrws', 'cfrwu', 'cfrnw', 'cfrns', 'cfrnu', 'cfrsw', 'cfrsn', 'cfrsu', 'cfruw', 'cfrun', 'cfrus', 'cfwrn', 'cfwrs', 'cfwru', 'cfwnr', 'cfwns', 'cfwnu', 'cfwsr', 'cfwsn', 'cfwsu', 'cfwur', 'cfwun', 'cfwus', 'cfnrw', 'cfnrs', 'cfnru', 'cfnwr', 'cfnws', 'cfnwu', 'cfnsr', 'cfnsw', 'cfnsu', 'cfnur', 'cfnuw', 'cfnus', 'cfsrw', 'cfsrn', 'cfsru', 'cfswr', 'cfswn', 'cfswu', 'cfsnr', 'cfsnw', 'cfsnu', 'cfsur', 'cfsuw', 'cfsun', 'cfurw', 'cfurn', 'cfurs', 'cfuwr', 'cfuwn', 'cfuws', 'cfunr', 'cfunw', 'cfuns', 'cfusr', 'cfusw', 'cfusn', 'csrwn', 'csrwf', 'csrwu', 'csrnw', 'csrnf', 'csrnu', 'csrfw', 'csrfn', 'csrfu', 'csruw', 'csrun', 'csruf', 'cswrn', 'cswrf', 'cswru', 'cswnr', 'cswnf', 'cswnu', 'cswfr', 'cswfn', 'cswfu', 'cswur', 'cswun', 'cswuf', 'csnrw', 'csnrf', 'csnru', 'csnwr', 'csnwf', 'csnwu', 'csnfr', 'csnfw', 'csnfu', 'csnur', 'csnuw', 'csnuf', 'csfrw', 'csfrn', 'csfru', 'csfwr', 'csfwn', 'csfwu', 'csfnr', 'csfnw', 'csfnu', 'csfur', 'csfuw', 'csfun', 'csurw', 'csurn', 'csurf', 'csuwr', 'csuwn', 'csuwf', 'csunr', 'csunw', 'csunf', 'csufr', 'csufw', 'csufn', 'curwn', 'curwf', 'curws', 'curnw', 'curnf', 'curns', 'curfw', 'curfn', 'curfs', 'cursw', 'cursn', 'cursf', 'cuwrn', 'cuwrf', 'cuwrs', 'cuwnr', 'cuwnf', 'cuwns', 'cuwfr', 'cuwfn', 'cuwfs', 'cuwsr', 'cuwsn', 'cuwsf', 'cunrw', 'cunrf', 'cunrs', 'cunwr', 'cunwf', 'cunws', 'cunfr', 'cunfw', 'cunfs', 'cunsr', 'cunsw', 'cunsf', 'cufrw', 'cufrn', 'cufrs', 'cufwr', 'cufwn', 'cufws', 'cufnr', 'cufnw', 'cufns', 'cufsr', 'cufsw', 'cufsn', 'cusrw', 'cusrn', 'cusrf', 'cuswr', 'cuswn', 'cuswf', 'cusnr', 'cusnw', 'cusnf', 'cusfr', 'cusfw', 'cusfn', 'nrwcf', 'nrwcs', 'nrwcu', 'nrwfc', 'nrwfs', 'nrwfu', 'nrwsc', 'nrwsf', 'nrwsu', 'nrwuc', 'nrwuf', 'nrwus', 'nrcwf', 'nrcws', 'nrcwu', 'nrcfw', 'nrcfs', 'nrcfu', 'nrcsw', 'nrcsf', 'nrcsu', 'nrcuw', 'nrcuf', 'nrcus', 'nrfwc', 'nrfws', 'nrfwu', 'nrfcw', 'nrfcs', 'nrfcu', 'nrfsw', 'nrfsc', 'nrfsu', 'nrfuw', 'nrfuc', 'nrfus', 'nrswc', 'nrswf', 'nrswu', 'nrscw', 'nrscf', 'nrscu', 'nrsfw', 'nrsfc', 'nrsfu', 'nrsuw', 'nrsuc', 'nrsuf', 'nruwc', 'nruwf', 'nruws', 'nrucw', 'nrucf', 'nrucs', 'nrufw', 'nrufc', 'nrufs', 'nrusw', 'nrusc', 'nrusf', 'nwrcf', 'nwrcs', 'nwrcu', 'nwrfc', 'nwrfs', 'nwrfu', 'nwrsc', 'nwrsf', 'nwrsu', 'nwruc', 'nwruf', 'nwrus', 'nwcrf', 'nwcrs', 'nwcru', 'nwcfr', 'nwcfs', 'nwcfu', 'nwcsr', 'nwcsf', 'nwcsu', 'nwcur', 'nwcuf', 'nwcus', 'nwfrc', 'nwfrs', 'nwfru', 'nwfcr', 'nwfcs', 'nwfcu', 'nwfsr', 'nwfsc', 'nwfsu', 'nwfur', 'nwfuc', 'nwfus', 'nwsrc', 'nwsrf', 'nwsru', 'nwscr', 'nwscf', 'nwscu', 'nwsfr', 'nwsfc', 'nwsfu', 'nwsur', 'nwsuc', 'nwsuf', 'nwurc', 'nwurf', 'nwurs', 'nwucr', 'nwucf', 'nwucs', 'nwufr', 'nwufc', 'nwufs', 'nwusr', 'nwusc', 'nwusf', 'ncrwf', 'ncrws', 'ncrwu', 'ncrfw', 'ncrfs', 'ncrfu', 'ncrsw', 'ncrsf', 'ncrsu', 'ncruw', 'ncruf', 'ncrus', 'ncwrf', 'ncwrs', 'ncwru', 'ncwfr', 'ncwfs', 'ncwfu', 'ncwsr', 'ncwsf', 'ncwsu', 'ncwur', 'ncwuf', 'ncwus', 'ncfrw', 'ncfrs', 'ncfru', 'ncfwr', 'ncfws', 'ncfwu', 'ncfsr', 'ncfsw', 'ncfsu', 'ncfur', 'ncfuw', 'ncfus', 'ncsrw', 'ncsrf', 'ncsru', 'ncswr', 'ncswf', 'ncswu', 'ncsfr', 'ncsfw', 'ncsfu', 'ncsur', 'ncsuw', 'ncsuf', 'ncurw', 'ncurf', 'ncurs', 'ncuwr', 'ncuwf', 'ncuws', 'ncufr', 'ncufw', 'ncufs', 'ncusr', 'ncusw', 'ncusf', 'nfrwc', 'nfrws', 'nfrwu', 'nfrcw', 'nfrcs', 'nfrcu', 'nfrsw', 'nfrsc', 'nfrsu', 'nfruw', 'nfruc', 'nfrus', 'nfwrc', 'nfwrs', 'nfwru', 'nfwcr', 'nfwcs', 'nfwcu', 'nfwsr', 'nfwsc', 'nfwsu', 'nfwur', 'nfwuc', 'nfwus', 'nfcrw', 'nfcrs', 'nfcru', 'nfcwr', 'nfcws', 'nfcwu', 'nfcsr', 'nfcsw', 'nfcsu', 'nfcur', 'nfcuw', 'nfcus', 'nfsrw', 'nfsrc', 'nfsru', 'nfswr', 'nfswc', 'nfswu', 'nfscr', 'nfscw', 'nfscu', 'nfsur', 'nfsuw', 'nfsuc', 'nfurw', 'nfurc', 'nfurs', 'nfuwr', 'nfuwc', 'nfuws', 'nfucr', 'nfucw', 'nfucs', 'nfusr', 'nfusw', 'nfusc', 'nsrwc', 'nsrwf', 'nsrwu', 'nsrcw', 'nsrcf', 'nsrcu', 'nsrfw', 'nsrfc', 'nsrfu', 'nsruw', 'nsruc', 'nsruf', 'nswrc', 'nswrf', 'nswru', 'nswcr', 'nswcf', 'nswcu', 'nswfr', 'nswfc', 'nswfu', 'nswur', 'nswuc', 'nswuf', 'nscrw', 'nscrf', 'nscru', 'nscwr', 'nscwf', 'nscwu', 'nscfr', 'nscfw', 'nscfu', 'nscur', 'nscuw', 'nscuf', 'nsfrw', 'nsfrc', 'nsfru', 'nsfwr', 'nsfwc', 'nsfwu', 'nsfcr', 'nsfcw', 'nsfcu', 'nsfur', 'nsfuw', 'nsfuc', 'nsurw', 'nsurc', 'nsurf', 'nsuwr', 'nsuwc', 'nsuwf', 'nsucr', 'nsucw', 'nsucf', 'nsufr', 'nsufw', 'nsufc', 'nurwc', 'nurwf', 'nurws', 'nurcw', 'nurcf', 'nurcs', 'nurfw', 'nurfc', 'nurfs', 'nursw', 'nursc', 'nursf', 'nuwrc', 'nuwrf', 'nuwrs', 'nuwcr', 'nuwcf', 'nuwcs', 'nuwfr', 'nuwfc', 'nuwfs', 'nuwsr', 'nuwsc', 'nuwsf', 'nucrw', 'nucrf', 'nucrs', 'nucwr', 'nucwf', 'nucws', 'nucfr', 'nucfw', 'nucfs', 'nucsr', 'nucsw', 'nucsf', 'nufrw', 'nufrc', 'nufrs', 'nufwr', 'nufwc', 'nufws', 'nufcr', 'nufcw', 'nufcs', 'nufsr', 'nufsw', 'nufsc', 'nusrw', 'nusrc', 'nusrf', 'nuswr', 'nuswc', 'nuswf', 'nuscr', 'nuscw', 'nuscf', 'nusfr', 'nusfw', 'nusfc', 'frwcn', 'frwcs', 'frwcu', 'frwnc', 'frwns', 'frwnu', 'frwsc', 'frwsn', 'frwsu', 'frwuc', 'frwun', 'frwus', 'frcwn', 'frcws', 'frcwu', 'frcnw', 'frcns', 'frcnu', 'frcsw', 'frcsn', 'frcsu', 'frcuw', 'frcun', 'frcus', 'frnwc', 'frnws', 'frnwu', 'frncw', 'frncs', 'frncu', 'frnsw', 'frnsc', 'frnsu', 'frnuw', 'frnuc', 'frnus', 'frswc', 'frswn', 'frswu', 'frscw', 'frscn', 'frscu', 'frsnw', 'frsnc', 'frsnu', 'frsuw', 'frsuc', 'frsun', 'fruwc', 'fruwn', 'fruws', 'frucw', 'frucn', 'frucs', 'frunw', 'frunc', 'fruns', 'frusw', 'frusc', 'frusn', 'fwrcn', 'fwrcs', 'fwrcu', 'fwrnc', 'fwrns', 'fwrnu', 'fwrsc', 'fwrsn', 'fwrsu', 'fwruc', 'fwrun', 'fwrus', 'fwcrn', 'fwcrs', 'fwcru', 'fwcnr', 'fwcns', 'fwcnu', 'fwcsr', 'fwcsn', 'fwcsu', 'fwcur', 'fwcun', 'fwcus', 'fwnrc', 'fwnrs', 'fwnru', 'fwncr', 'fwncs', 'fwncu', 'fwnsr', 'fwnsc', 'fwnsu', 'fwnur', 'fwnuc', 'fwnus', 'fwsrc', 'fwsrn', 'fwsru', 'fwscr', 'fwscn', 'fwscu', 'fwsnr', 'fwsnc', 'fwsnu', 'fwsur', 'fwsuc', 'fwsun', 'fwurc', 'fwurn', 'fwurs', 'fwucr', 'fwucn', 'fwucs', 'fwunr', 'fwunc', 'fwuns', 'fwusr', 'fwusc', 'fwusn', 'fcrwn', 'fcrws', 'fcrwu', 'fcrnw', 'fcrns', 'fcrnu', 'fcrsw', 'fcrsn', 'fcrsu', 'fcruw', 'fcrun', 'fcrus', 'fcwrn', 'fcwrs', 'fcwru', 'fcwnr', 'fcwns', 'fcwnu', 'fcwsr', 'fcwsn', 'fcwsu', 'fcwur', 'fcwun', 'fcwus', 'fcnrw', 'fcnrs', 'fcnru', 'fcnwr', 'fcnws', 'fcnwu', 'fcnsr', 'fcnsw', 'fcnsu', 'fcnur', 'fcnuw', 'fcnus', 'fcsrw', 'fcsrn', 'fcsru', 'fcswr', 'fcswn', 'fcswu', 'fcsnr', 'fcsnw', 'fcsnu', 'fcsur', 'fcsuw', 'fcsun', 'fcurw', 'fcurn', 'fcurs', 'fcuwr', 'fcuwn', 'fcuws', 'fcunr', 'fcunw', 'fcuns', 'fcusr', 'fcusw', 'fcusn', 'fnrwc', 'fnrws', 'fnrwu', 'fnrcw', 'fnrcs', 'fnrcu', 'fnrsw', 'fnrsc', 'fnrsu', 'fnruw', 'fnruc', 'fnrus', 'fnwrc', 'fnwrs', 'fnwru', 'fnwcr', 'fnwcs', 'fnwcu', 'fnwsr', 'fnwsc', 'fnwsu', 'fnwur', 'fnwuc', 'fnwus', 'fncrw', 'fncrs', 'fncru', 'fncwr', 'fncws', 'fncwu', 'fncsr', 'fncsw', 'fncsu', 'fncur', 'fncuw', 'fncus', 'fnsrw', 'fnsrc', 'fnsru', 'fnswr', 'fnswc', 'fnswu', 'fnscr', 'fnscw', 'fnscu', 'fnsur', 'fnsuw', 'fnsuc', 'fnurw', 'fnurc', 'fnurs', 'fnuwr', 'fnuwc', 'fnuws', 'fnucr', 'fnucw', 'fnucs', 'fnusr', 'fnusw', 'fnusc', 'fsrwc', 'fsrwn', 'fsrwu', 'fsrcw', 'fsrcn', 'fsrcu', 'fsrnw', 'fsrnc', 'fsrnu', 'fsruw', 'fsruc', 'fsrun', 'fswrc', 'fswrn', 'fswru', 'fswcr', 'fswcn', 'fswcu', 'fswnr', 'fswnc', 'fswnu', 'fswur', 'fswuc', 'fswun', 'fscrw', 'fscrn', 'fscru', 'fscwr', 'fscwn', 'fscwu', 'fscnr', 'fscnw', 'fscnu', 'fscur', 'fscuw', 'fscun', 'fsnrw', 'fsnrc', 'fsnru', 'fsnwr', 'fsnwc', 'fsnwu', 'fsncr', 'fsncw', 'fsncu', 'fsnur', 'fsnuw', 'fsnuc', 'fsurw', 'fsurc', 'fsurn', 'fsuwr', 'fsuwc', 'fsuwn', 'fsucr', 'fsucw', 'fsucn', 'fsunr', 'fsunw', 'fsunc', 'furwc', 'furwn', 'furws', 'furcw', 'furcn', 'furcs', 'furnw', 'furnc', 'furns', 'fursw', 'fursc', 'fursn', 'fuwrc', 'fuwrn', 'fuwrs', 'fuwcr', 'fuwcn', 'fuwcs', 'fuwnr', 'fuwnc', 'fuwns', 'fuwsr', 'fuwsc', 'fuwsn', 'fucrw', 'fucrn', 'fucrs', 'fucwr', 'fucwn', 'fucws', 'fucnr', 'fucnw', 'fucns', 'fucsr', 'fucsw', 'fucsn', 'funrw', 'funrc', 'funrs', 'funwr', 'funwc', 'funws', 'funcr', 'funcw', 'funcs', 'funsr', 'funsw', 'funsc', 'fusrw', 'fusrc', 'fusrn', 'fuswr', 'fuswc', 'fuswn', 'fuscr', 'fuscw', 'fuscn', 'fusnr', 'fusnw', 'fusnc', 'srwcn', 'srwcf', 'srwcu', 'srwnc', 'srwnf', 'srwnu', 'srwfc', 'srwfn', 'srwfu', 'srwuc', 'srwun', 'srwuf', 'srcwn', 'srcwf', 'srcwu', 'srcnw', 'srcnf', 'srcnu', 'srcfw', 'srcfn', 'srcfu', 'srcuw', 'srcun', 'srcuf', 'srnwc', 'srnwf', 'srnwu', 'srncw', 'srncf', 'srncu', 'srnfw', 'srnfc', 'srnfu', 'srnuw', 'srnuc', 'srnuf', 'srfwc', 'srfwn', 'srfwu', 'srfcw', 'srfcn', 'srfcu', 'srfnw', 'srfnc', 'srfnu', 'srfuw', 'srfuc', 'srfun', 'sruwc', 'sruwn', 'sruwf', 'srucw', 'srucn', 'srucf', 'srunw', 'srunc', 'srunf', 'srufw', 'srufc', 'srufn', 'swrcn', 'swrcf', 'swrcu', 'swrnc', 'swrnf', 'swrnu', 'swrfc', 'swrfn', 'swrfu', 'swruc', 'swrun', 'swruf', 'swcrn', 'swcrf', 'swcru', 'swcnr', 'swcnf', 'swcnu', 'swcfr', 'swcfn', 'swcfu', 'swcur', 'swcun', 'swcuf', 'swnrc', 'swnrf', 'swnru', 'swncr', 'swncf', 'swncu', 'swnfr', 'swnfc', 'swnfu', 'swnur', 'swnuc', 'swnuf', 'swfrc', 'swfrn', 'swfru', 'swfcr', 'swfcn', 'swfcu', 'swfnr', 'swfnc', 'swfnu', 'swfur', 'swfuc', 'swfun', 'swurc', 'swurn', 'swurf', 'swucr', 'swucn', 'swucf', 'swunr', 'swunc', 'swunf', 'swufr', 'swufc', 'swufn', 'scrwn', 'scrwf', 'scrwu', 'scrnw', 'scrnf', 'scrnu', 'scrfw', 'scrfn', 'scrfu', 'scruw', 'scrun', 'scruf', 'scwrn', 'scwrf', 'scwru', 'scwnr', 'scwnf', 'scwnu', 'scwfr', 'scwfn', 'scwfu', 'scwur', 'scwun', 'scwuf', 'scnrw', 'scnrf', 'scnru', 'scnwr', 'scnwf', 'scnwu', 'scnfr', 'scnfw', 'scnfu', 'scnur', 'scnuw', 'scnuf', 'scfrw', 'scfrn', 'scfru', 'scfwr', 'scfwn', 'scfwu', 'scfnr', 'scfnw', 'scfnu', 'scfur', 'scfuw', 'scfun', 'scurw', 'scurn', 'scurf', 'scuwr', 'scuwn', 'scuwf', 'scunr', 'scunw', 'scunf', 'scufr', 'scufw', 'scufn', 'snrwc', 'snrwf', 'snrwu', 'snrcw', 'snrcf', 'snrcu', 'snrfw', 'snrfc', 'snrfu', 'snruw', 'snruc', 'snruf', 'snwrc', 'snwrf', 'snwru', 'snwcr', 'snwcf', 'snwcu', 'snwfr', 'snwfc', 'snwfu', 'snwur', 'snwuc', 'snwuf', 'sncrw', 'sncrf', 'sncru', 'sncwr', 'sncwf', 'sncwu', 'sncfr', 'sncfw', 'sncfu', 'sncur', 'sncuw', 'sncuf', 'snfrw', 'snfrc', 'snfru', 'snfwr', 'snfwc', 'snfwu', 'snfcr', 'snfcw', 'snfcu', 'snfur', 'snfuw', 'snfuc', 'snurw', 'snurc', 'snurf', 'snuwr', 'snuwc', 'snuwf', 'snucr', 'snucw', 'snucf', 'snufr', 'snufw', 'snufc', 'sfrwc', 'sfrwn', 'sfrwu', 'sfrcw', 'sfrcn', 'sfrcu', 'sfrnw', 'sfrnc', 'sfrnu', 'sfruw', 'sfruc', 'sfrun', 'sfwrc', 'sfwrn', 'sfwru', 'sfwcr', 'sfwcn', 'sfwcu', 'sfwnr', 'sfwnc', 'sfwnu', 'sfwur', 'sfwuc', 'sfwun', 'sfcrw', 'sfcrn', 'sfcru', 'sfcwr', 'sfcwn', 'sfcwu', 'sfcnr', 'sfcnw', 'sfcnu', 'sfcur', 'sfcuw', 'sfcun', 'sfnrw', 'sfnrc', 'sfnru', 'sfnwr', 'sfnwc', 'sfnwu', 'sfncr', 'sfncw', 'sfncu', 'sfnur', 'sfnuw', 'sfnuc', 'sfurw', 'sfurc', 'sfurn', 'sfuwr', 'sfuwc', 'sfuwn', 'sfucr', 'sfucw', 'sfucn', 'sfunr', 'sfunw', 'sfunc', 'surwc', 'surwn', 'surwf', 'surcw', 'surcn', 'surcf', 'surnw', 'surnc', 'surnf', 'surfw', 'surfc', 'surfn', 'suwrc', 'suwrn', 'suwrf', 'suwcr', 'suwcn', 'suwcf', 'suwnr', 'suwnc', 'suwnf', 'suwfr', 'suwfc', 'suwfn', 'sucrw', 'sucrn', 'sucrf', 'sucwr', 'sucwn', 'sucwf', 'sucnr', 'sucnw', 'sucnf', 'sucfr', 'sucfw', 'sucfn', 'sunrw', 'sunrc', 'sunrf', 'sunwr', 'sunwc', 'sunwf', 'suncr', 'suncw', 'suncf', 'sunfr', 'sunfw', 'sunfc', 'sufrw', 'sufrc', 'sufrn', 'sufwr', 'sufwc', 'sufwn', 'sufcr', 'sufcw', 'sufcn', 'sufnr', 'sufnw', 'sufnc', 'urwcn', 'urwcf', 'urwcs', 'urwnc', 'urwnf', 'urwns', 'urwfc', 'urwfn', 'urwfs', 'urwsc', 'urwsn', 'urwsf', 'urcwn', 'urcwf', 'urcws', 'urcnw', 'urcnf', 'urcns', 'urcfw', 'urcfn', 'urcfs', 'urcsw', 'urcsn', 'urcsf', 'urnwc', 'urnwf', 'urnws', 'urncw', 'urncf', 'urncs', 'urnfw', 'urnfc', 'urnfs', 'urnsw', 'urnsc', 'urnsf', 'urfwc', 'urfwn', 'urfws', 'urfcw', 'urfcn', 'urfcs', 'urfnw', 'urfnc', 'urfns', 'urfsw', 'urfsc', 'urfsn', 'urswc', 'urswn', 'urswf', 'urscw', 'urscn', 'urscf', 'ursnw', 'ursnc', 'ursnf', 'ursfw', 'ursfc', 'ursfn', 'uwrcn', 'uwrcf', 'uwrcs', 'uwrnc', 'uwrnf', 'uwrns', 'uwrfc', 'uwrfn', 'uwrfs', 'uwrsc', 'uwrsn', 'uwrsf', 'uwcrn', 'uwcrf', 'uwcrs', 'uwcnr', 'uwcnf', 'uwcns', 'uwcfr', 'uwcfn', 'uwcfs', 'uwcsr', 'uwcsn', 'uwcsf', 'uwnrc', 'uwnrf', 'uwnrs', 'uwncr', 'uwncf', 'uwncs', 'uwnfr', 'uwnfc', 'uwnfs', 'uwnsr', 'uwnsc', 'uwnsf', 'uwfrc', 'uwfrn', 'uwfrs', 'uwfcr', 'uwfcn', 'uwfcs', 'uwfnr', 'uwfnc', 'uwfns', 'uwfsr', 'uwfsc', 'uwfsn', 'uwsrc', 'uwsrn', 'uwsrf', 'uwscr', 'uwscn', 'uwscf', 'uwsnr', 'uwsnc', 'uwsnf', 'uwsfr', 'uwsfc', 'uwsfn', 'ucrwn', 'ucrwf', 'ucrws', 'ucrnw', 'ucrnf', 'ucrns', 'ucrfw', 'ucrfn', 'ucrfs', 'ucrsw', 'ucrsn', 'ucrsf', 'ucwrn', 'ucwrf', 'ucwrs', 'ucwnr', 'ucwnf', 'ucwns', 'ucwfr', 'ucwfn', 'ucwfs', 'ucwsr', 'ucwsn', 'ucwsf', 'ucnrw', 'ucnrf', 'ucnrs', 'ucnwr', 'ucnwf', 'ucnws', 'ucnfr', 'ucnfw', 'ucnfs', 'ucnsr', 'ucnsw', 'ucnsf', 'ucfrw', 'ucfrn', 'ucfrs', 'ucfwr', 'ucfwn', 'ucfws', 'ucfnr', 'ucfnw', 'ucfns', 'ucfsr', 'ucfsw', 'ucfsn', 'ucsrw', 'ucsrn', 'ucsrf', 'ucswr', 'ucswn', 'ucswf', 'ucsnr', 'ucsnw', 'ucsnf', 'ucsfr', 'ucsfw', 'ucsfn', 'unrwc', 'unrwf', 'unrws', 'unrcw', 'unrcf', 'unrcs', 'unrfw', 'unrfc', 'unrfs', 'unrsw', 'unrsc', 'unrsf', 'unwrc', 'unwrf', 'unwrs', 'unwcr', 'unwcf', 'unwcs', 'unwfr', 'unwfc', 'unwfs', 'unwsr', 'unwsc', 'unwsf', 'uncrw', 'uncrf', 'uncrs', 'uncwr', 'uncwf', 'uncws', 'uncfr', 'uncfw', 'uncfs', 'uncsr', 'uncsw', 'uncsf', 'unfrw', 'unfrc', 'unfrs', 'unfwr', 'unfwc', 'unfws', 'unfcr', 'unfcw', 'unfcs', 'unfsr', 'unfsw', 'unfsc', 'unsrw', 'unsrc', 'unsrf', 'unswr', 'unswc', 'unswf', 'unscr', 'unscw', 'unscf', 'unsfr', 'unsfw', 'unsfc', 'ufrwc', 'ufrwn', 'ufrws', 'ufrcw', 'ufrcn', 'ufrcs', 'ufrnw', 'ufrnc', 'ufrns', 'ufrsw', 'ufrsc', 'ufrsn', 'ufwrc', 'ufwrn', 'ufwrs', 'ufwcr', 'ufwcn', 'ufwcs', 'ufwnr', 'ufwnc', 'ufwns', 'ufwsr', 'ufwsc', 'ufwsn', 'ufcrw', 'ufcrn', 'ufcrs', 'ufcwr', 'ufcwn', 'ufcws', 'ufcnr', 'ufcnw', 'ufcns', 'ufcsr', 'ufcsw', 'ufcsn', 'ufnrw', 'ufnrc', 'ufnrs', 'ufnwr', 'ufnwc', 'ufnws', 'ufncr', 'ufncw', 'ufncs', 'ufnsr', 'ufnsw', 'ufnsc', 'ufsrw', 'ufsrc', 'ufsrn', 'ufswr', 'ufswc', 'ufswn', 'ufscr', 'ufscw', 'ufscn', 'ufsnr', 'ufsnw', 'ufsnc', 'usrwc', 'usrwn', 'usrwf', 'usrcw', 'usrcn', 'usrcf', 'usrnw', 'usrnc', 'usrnf', 'usrfw', 'usrfc', 'usrfn', 'uswrc', 'uswrn', 'uswrf', 'uswcr', 'uswcn', 'uswcf', 'uswnr', 'uswnc', 'uswnf', 'uswfr', 'uswfc', 'uswfn', 'uscrw', 'uscrn', 'uscrf', 'uscwr', 'uscwn', 'uscwf', 'uscnr', 'uscnw', 'uscnf', 'uscfr', 'uscfw', 'uscfn', 'usnrw', 'usnrc', 'usnrf', 'usnwr', 'usnwc', 'usnwf', 'usncr', 'usncw', 'usncf', 'usnfr', 'usnfw', 'usnfc', 'usfrw', 'usfrc', 'usfrn', 'usfwr', 'usfwc', 'usfwn', 'usfcr', 'usfcw', 'usfcn', 'usfnr', 'usfnw', 'usfnc', 'rwcnfs', 'rwcnfu', 'rwcnsf', 'rwcnsu', 'rwcnuf', 'rwcnus', 'rwcfns', 'rwcfnu', 'rwcfsn', 'rwcfsu', 'rwcfun', 'rwcfus', 'rwcsnf', 'rwcsnu', 'rwcsfn', 'rwcsfu', 'rwcsun', 'rwcsuf', 'rwcunf', 'rwcuns', 'rwcufn', 'rwcufs', 'rwcusn', 'rwcusf', 'rwncfs', 'rwncfu', 'rwncsf', 'rwncsu', 'rwncuf', 'rwncus', 'rwnfcs', 'rwnfcu', 'rwnfsc', 'rwnfsu', 'rwnfuc', 'rwnfus', 'rwnscf', 'rwnscu', 'rwnsfc', 'rwnsfu', 'rwnsuc', 'rwnsuf', 'rwnucf', 'rwnucs', 'rwnufc', 'rwnufs', 'rwnusc', 'rwnusf', 'rwfcns', 'rwfcnu', 'rwfcsn', 'rwfcsu', 'rwfcun', 'rwfcus', 'rwfncs', 'rwfncu', 'rwfnsc', 'rwfnsu', 'rwfnuc', 'rwfnus', 'rwfscn', 'rwfscu', 'rwfsnc', 'rwfsnu', 'rwfsuc', 'rwfsun', 'rwfucn', 'rwfucs', 'rwfunc', 'rwfuns', 'rwfusc', 'rwfusn', 'rwscnf', 'rwscnu', 'rwscfn', 'rwscfu', 'rwscun', 'rwscuf', 'rwsncf', 'rwsncu', 'rwsnfc', 'rwsnfu', 'rwsnuc', 'rwsnuf', 'rwsfcn', 'rwsfcu', 'rwsfnc', 'rwsfnu', 'rwsfuc', 'rwsfun', 'rwsucn', 'rwsucf', 'rwsunc', 'rwsunf', 'rwsufc', 'rwsufn', 'rwucnf', 'rwucns', 'rwucfn', 'rwucfs', 'rwucsn', 'rwucsf', 'rwuncf', 'rwuncs', 'rwunfc', 'rwunfs', 'rwunsc', 'rwunsf', 'rwufcn', 'rwufcs', 'rwufnc', 'rwufns', 'rwufsc', 'rwufsn', 'rwuscn', 'rwuscf', 'rwusnc', 'rwusnf', 'rwusfc', 'rwusfn', 'rcwnfs', 'rcwnfu', 'rcwnsf', 'rcwnsu', 'rcwnuf', 'rcwnus', 'rcwfns', 'rcwfnu', 'rcwfsn', 'rcwfsu', 'rcwfun', 'rcwfus', 'rcwsnf', 'rcwsnu', 'rcwsfn', 'rcwsfu', 'rcwsun', 'rcwsuf', 'rcwunf', 'rcwuns', 'rcwufn', 'rcwufs', 'rcwusn', 'rcwusf', 'rcnwfs', 'rcnwfu', 'rcnwsf', 'rcnwsu', 'rcnwuf', 'rcnwus', 'rcnfws', 'rcnfwu', 'rcnfsw', 'rcnfsu', 'rcnfuw', 'rcnfus', 'rcnswf', 'rcnswu', 'rcnsfw', 'rcnsfu', 'rcnsuw', 'rcnsuf', 'rcnuwf', 'rcnuws', 'rcnufw', 'rcnufs', 'rcnusw', 'rcnusf', 'rcfwns', 'rcfwnu', 'rcfwsn', 'rcfwsu', 'rcfwun', 'rcfwus', 'rcfnws', 'rcfnwu', 'rcfnsw', 'rcfnsu', 'rcfnuw', 'rcfnus', 'rcfswn', 'rcfswu', 'rcfsnw', 'rcfsnu', 'rcfsuw', 'rcfsun', 'rcfuwn', 'rcfuws', 'rcfunw', 'rcfuns', 'rcfusw', 'rcfusn', 'rcswnf', 'rcswnu', 'rcswfn', 'rcswfu', 'rcswun', 'rcswuf', 'rcsnwf', 'rcsnwu', 'rcsnfw', 'rcsnfu', 'rcsnuw', 'rcsnuf', 'rcsfwn', 'rcsfwu', 'rcsfnw', 'rcsfnu', 'rcsfuw', 'rcsfun', 'rcsuwn', 'rcsuwf', 'rcsunw', 'rcsunf', 'rcsufw', 'rcsufn', 'rcuwnf', 'rcuwns', 'rcuwfn', 'rcuwfs', 'rcuwsn', 'rcuwsf', 'rcunwf', 'rcunws', 'rcunfw', 'rcunfs', 'rcunsw', 'rcunsf', 'rcufwn', 'rcufws', 'rcufnw', 'rcufns', 'rcufsw', 'rcufsn', 'rcuswn', 'rcuswf', 'rcusnw', 'rcusnf', 'rcusfw', 'rcusfn', 'rnwcfs', 'rnwcfu', 'rnwcsf', 'rnwcsu', 'rnwcuf', 'rnwcus', 'rnwfcs', 'rnwfcu', 'rnwfsc', 'rnwfsu', 'rnwfuc', 'rnwfus', 'rnwscf', 'rnwscu', 'rnwsfc', 'rnwsfu', 'rnwsuc', 'rnwsuf', 'rnwucf', 'rnwucs', 'rnwufc', 'rnwufs', 'rnwusc', 'rnwusf', 'rncwfs', 'rncwfu', 'rncwsf', 'rncwsu', 'rncwuf', 'rncwus', 'rncfws', 'rncfwu', 'rncfsw', 'rncfsu', 'rncfuw', 'rncfus', 'rncswf', 'rncswu', 'rncsfw', 'rncsfu', 'rncsuw', 'rncsuf', 'rncuwf', 'rncuws', 'rncufw', 'rncufs', 'rncusw', 'rncusf', 'rnfwcs', 'rnfwcu', 'rnfwsc', 'rnfwsu', 'rnfwuc', 'rnfwus', 'rnfcws', 'rnfcwu', 'rnfcsw', 'rnfcsu', 'rnfcuw', 'rnfcus', 'rnfswc', 'rnfswu', 'rnfscw', 'rnfscu', 'rnfsuw', 'rnfsuc', 'rnfuwc', 'rnfuws', 'rnfucw', 'rnfucs', 'rnfusw', 'rnfusc', 'rnswcf', 'rnswcu', 'rnswfc', 'rnswfu', 'rnswuc', 'rnswuf', 'rnscwf', 'rnscwu', 'rnscfw', 'rnscfu', 'rnscuw', 'rnscuf', 'rnsfwc', 'rnsfwu', 'rnsfcw', 'rnsfcu', 'rnsfuw', 'rnsfuc', 'rnsuwc', 'rnsuwf', 'rnsucw', 'rnsucf', 'rnsufw', 'rnsufc', 'rnuwcf', 'rnuwcs', 'rnuwfc', 'rnuwfs', 'rnuwsc', 'rnuwsf', 'rnucwf', 'rnucws', 'rnucfw', 'rnucfs', 'rnucsw', 'rnucsf', 'rnufwc', 'rnufws', 'rnufcw', 'rnufcs', 'rnufsw', 'rnufsc', 'rnuswc', 'rnuswf', 'rnuscw', 'rnuscf', 'rnusfw', 'rnusfc', 'rfwcns', 'rfwcnu', 'rfwcsn', 'rfwcsu', 'rfwcun', 'rfwcus', 'rfwncs', 'rfwncu', 'rfwnsc', 'rfwnsu', 'rfwnuc', 'rfwnus', 'rfwscn', 'rfwscu', 'rfwsnc', 'rfwsnu', 'rfwsuc', 'rfwsun', 'rfwucn', 'rfwucs', 'rfwunc', 'rfwuns', 'rfwusc', 'rfwusn', 'rfcwns', 'rfcwnu', 'rfcwsn', 'rfcwsu', 'rfcwun', 'rfcwus', 'rfcnws', 'rfcnwu', 'rfcnsw', 'rfcnsu', 'rfcnuw', 'rfcnus', 'rfcswn', 'rfcswu', 'rfcsnw', 'rfcsnu', 'rfcsuw', 'rfcsun', 'rfcuwn', 'rfcuws', 'rfcunw', 'rfcuns', 'rfcusw', 'rfcusn', 'rfnwcs', 'rfnwcu', 'rfnwsc', 'rfnwsu', 'rfnwuc', 'rfnwus', 'rfncws', 'rfncwu', 'rfncsw', 'rfncsu', 'rfncuw', 'rfncus', 'rfnswc', 'rfnswu', 'rfnscw', 'rfnscu', 'rfnsuw', 'rfnsuc', 'rfnuwc', 'rfnuws', 'rfnucw', 'rfnucs', 'rfnusw', 'rfnusc', 'rfswcn', 'rfswcu', 'rfswnc', 'rfswnu', 'rfswuc', 'rfswun', 'rfscwn', 'rfscwu', 'rfscnw', 'rfscnu', 'rfscuw', 'rfscun', 'rfsnwc', 'rfsnwu', 'rfsncw', 'rfsncu', 'rfsnuw', 'rfsnuc', 'rfsuwc', 'rfsuwn', 'rfsucw', 'rfsucn', 'rfsunw', 'rfsunc', 'rfuwcn', 'rfuwcs', 'rfuwnc', 'rfuwns', 'rfuwsc', 'rfuwsn', 'rfucwn', 'rfucws', 'rfucnw', 'rfucns', 'rfucsw', 'rfucsn', 'rfunwc', 'rfunws', 'rfuncw', 'rfuncs', 'rfunsw', 'rfunsc', 'rfuswc', 'rfuswn', 'rfuscw', 'rfuscn', 'rfusnw', 'rfusnc', 'rswcnf', 'rswcnu', 'rswcfn', 'rswcfu', 'rswcun', 'rswcuf', 'rswncf', 'rswncu', 'rswnfc', 'rswnfu', 'rswnuc', 'rswnuf', 'rswfcn', 'rswfcu', 'rswfnc', 'rswfnu', 'rswfuc', 'rswfun', 'rswucn', 'rswucf', 'rswunc', 'rswunf', 'rswufc', 'rswufn', 'rscwnf', 'rscwnu', 'rscwfn', 'rscwfu', 'rscwun', 'rscwuf', 'rscnwf', 'rscnwu', 'rscnfw', 'rscnfu', 'rscnuw', 'rscnuf', 'rscfwn', 'rscfwu', 'rscfnw', 'rscfnu', 'rscfuw', 'rscfun', 'rscuwn', 'rscuwf', 'rscunw', 'rscunf', 'rscufw', 'rscufn', 'rsnwcf', 'rsnwcu', 'rsnwfc', 'rsnwfu', 'rsnwuc', 'rsnwuf', 'rsncwf', 'rsncwu', 'rsncfw', 'rsncfu', 'rsncuw', 'rsncuf', 'rsnfwc', 'rsnfwu', 'rsnfcw', 'rsnfcu', 'rsnfuw', 'rsnfuc', 'rsnuwc', 'rsnuwf', 'rsnucw', 'rsnucf', 'rsnufw', 'rsnufc', 'rsfwcn', 'rsfwcu', 'rsfwnc', 'rsfwnu', 'rsfwuc', 'rsfwun', 'rsfcwn', 'rsfcwu', 'rsfcnw', 'rsfcnu', 'rsfcuw', 'rsfcun', 'rsfnwc', 'rsfnwu', 'rsfncw', 'rsfncu', 'rsfnuw', 'rsfnuc', 'rsfuwc', 'rsfuwn', 'rsfucw', 'rsfucn', 'rsfunw', 'rsfunc', 'rsuwcn', 'rsuwcf', 'rsuwnc', 'rsuwnf', 'rsuwfc', 'rsuwfn', 'rsucwn', 'rsucwf', 'rsucnw', 'rsucnf', 'rsucfw', 'rsucfn', 'rsunwc', 'rsunwf', 'rsuncw', 'rsuncf', 'rsunfw', 'rsunfc', 'rsufwc', 'rsufwn', 'rsufcw', 'rsufcn', 'rsufnw', 'rsufnc', 'ruwcnf', 'ruwcns', 'ruwcfn', 'ruwcfs', 'ruwcsn', 'ruwcsf', 'ruwncf', 'ruwncs', 'ruwnfc', 'ruwnfs', 'ruwnsc', 'ruwnsf', 'ruwfcn', 'ruwfcs', 'ruwfnc', 'ruwfns', 'ruwfsc', 'ruwfsn', 'ruwscn', 'ruwscf', 'ruwsnc', 'ruwsnf', 'ruwsfc', 'ruwsfn', 'rucwnf', 'rucwns', 'rucwfn', 'rucwfs', 'rucwsn', 'rucwsf', 'rucnwf', 'rucnws', 'rucnfw', 'rucnfs', 'rucnsw', 'rucnsf', 'rucfwn', 'rucfws', 'rucfnw', 'rucfns', 'rucfsw', 'rucfsn', 'rucswn', 'rucswf', 'rucsnw', 'rucsnf', 'rucsfw', 'rucsfn', 'runwcf', 'runwcs', 'runwfc', 'runwfs', 'runwsc', 'runwsf', 'runcwf', 'runcws', 'runcfw', 'runcfs', 'runcsw', 'runcsf', 'runfwc', 'runfws', 'runfcw', 'runfcs', 'runfsw', 'runfsc', 'runswc', 'runswf', 'runscw', 'runscf', 'runsfw', 'runsfc', 'rufwcn', 'rufwcs', 'rufwnc', 'rufwns', 'rufwsc', 'rufwsn', 'rufcwn', 'rufcws', 'rufcnw', 'rufcns', 'rufcsw', 'rufcsn', 'rufnwc', 'rufnws', 'rufncw', 'rufncs', 'rufnsw', 'rufnsc', 'rufswc', 'rufswn', 'rufscw', 'rufscn', 'rufsnw', 'rufsnc', 'ruswcn', 'ruswcf', 'ruswnc', 'ruswnf', 'ruswfc', 'ruswfn', 'ruscwn', 'ruscwf', 'ruscnw', 'ruscnf', 'ruscfw', 'ruscfn', 'rusnwc', 'rusnwf', 'rusncw', 'rusncf', 'rusnfw', 'rusnfc', 'rusfwc', 'rusfwn', 'rusfcw', 'rusfcn', 'rusfnw', 'rusfnc', 'wrcnfs', 'wrcnfu', 'wrcnsf', 'wrcnsu', 'wrcnuf', 'wrcnus', 'wrcfns', 'wrcfnu', 'wrcfsn', 'wrcfsu', 'wrcfun', 'wrcfus', 'wrcsnf', 'wrcsnu', 'wrcsfn', 'wrcsfu', 'wrcsun', 'wrcsuf', 'wrcunf', 'wrcuns', 'wrcufn', 'wrcufs', 'wrcusn', 'wrcusf', 'wrncfs', 'wrncfu', 'wrncsf', 'wrncsu', 'wrncuf', 'wrncus', 'wrnfcs', 'wrnfcu', 'wrnfsc', 'wrnfsu', 'wrnfuc', 'wrnfus', 'wrnscf', 'wrnscu', 'wrnsfc', 'wrnsfu', 'wrnsuc', 'wrnsuf', 'wrnucf', 'wrnucs', 'wrnufc', 'wrnufs', 'wrnusc', 'wrnusf', 'wrfcns', 'wrfcnu', 'wrfcsn', 'wrfcsu', 'wrfcun', 'wrfcus', 'wrfncs', 'wrfncu', 'wrfnsc', 'wrfnsu', 'wrfnuc', 'wrfnus', 'wrfscn', 'wrfscu', 'wrfsnc', 'wrfsnu', 'wrfsuc', 'wrfsun', 'wrfucn', 'wrfucs', 'wrfunc', 'wrfuns', 'wrfusc', 'wrfusn', 'wrscnf', 'wrscnu', 'wrscfn', 'wrscfu', 'wrscun', 'wrscuf', 'wrsncf', 'wrsncu', 'wrsnfc', 'wrsnfu', 'wrsnuc', 'wrsnuf', 'wrsfcn', 'wrsfcu', 'wrsfnc', 'wrsfnu', 'wrsfuc', 'wrsfun', 'wrsucn', 'wrsucf', 'wrsunc', 'wrsunf', 'wrsufc', 'wrsufn', 'wrucnf', 'wrucns', 'wrucfn', 'wrucfs', 'wrucsn', 'wrucsf', 'wruncf', 'wruncs', 'wrunfc', 'wrunfs', 'wrunsc', 'wrunsf', 'wrufcn', 'wrufcs', 'wrufnc', 'wrufns', 'wrufsc', 'wrufsn', 'wruscn', 'wruscf', 'wrusnc', 'wrusnf', 'wrusfc', 'wrusfn', 'wcrnfs', 'wcrnfu', 'wcrnsf', 'wcrnsu', 'wcrnuf', 'wcrnus', 'wcrfns', 'wcrfnu', 'wcrfsn', 'wcrfsu', 'wcrfun', 'wcrfus', 'wcrsnf', 'wcrsnu', 'wcrsfn', 'wcrsfu', 'wcrsun', 'wcrsuf', 'wcrunf', 'wcruns', 'wcrufn', 'wcrufs', 'wcrusn', 'wcrusf', 'wcnrfs', 'wcnrfu', 'wcnrsf', 'wcnrsu', 'wcnruf', 'wcnrus', 'wcnfrs', 'wcnfru', 'wcnfsr', 'wcnfsu', 'wcnfur', 'wcnfus', 'wcnsrf', 'wcnsru', 'wcnsfr', 'wcnsfu', 'wcnsur', 'wcnsuf', 'wcnurf', 'wcnurs', 'wcnufr', 'wcnufs', 'wcnusr', 'wcnusf', 'wcfrns', 'wcfrnu', 'wcfrsn', 'wcfrsu', 'wcfrun', 'wcfrus', 'wcfnrs', 'wcfnru', 'wcfnsr', 'wcfnsu', 'wcfnur', 'wcfnus', 'wcfsrn', 'wcfsru', 'wcfsnr', 'wcfsnu', 'wcfsur', 'wcfsun', 'wcfurn', 'wcfurs', 'wcfunr', 'wcfuns', 'wcfusr', 'wcfusn', 'wcsrnf', 'wcsrnu', 'wcsrfn', 'wcsrfu', 'wcsrun', 'wcsruf', 'wcsnrf', 'wcsnru', 'wcsnfr', 'wcsnfu', 'wcsnur', 'wcsnuf', 'wcsfrn', 'wcsfru', 'wcsfnr', 'wcsfnu', 'wcsfur', 'wcsfun', 'wcsurn', 'wcsurf', 'wcsunr', 'wcsunf', 'wcsufr', 'wcsufn', 'wcurnf', 'wcurns', 'wcurfn', 'wcurfs', 'wcursn', 'wcursf', 'wcunrf', 'wcunrs', 'wcunfr', 'wcunfs', 'wcunsr', 'wcunsf', 'wcufrn', 'wcufrs', 'wcufnr', 'wcufns', 'wcufsr', 'wcufsn', 'wcusrn', 'wcusrf', 'wcusnr', 'wcusnf', 'wcusfr', 'wcusfn', 'wnrcfs', 'wnrcfu', 'wnrcsf', 'wnrcsu', 'wnrcuf', 'wnrcus', 'wnrfcs', 'wnrfcu', 'wnrfsc', 'wnrfsu', 'wnrfuc', 'wnrfus', 'wnrscf', 'wnrscu', 'wnrsfc', 'wnrsfu', 'wnrsuc', 'wnrsuf', 'wnrucf', 'wnrucs', 'wnrufc', 'wnrufs', 'wnrusc', 'wnrusf', 'wncrfs', 'wncrfu', 'wncrsf', 'wncrsu', 'wncruf', 'wncrus', 'wncfrs', 'wncfru', 'wncfsr', 'wncfsu', 'wncfur', 'wncfus', 'wncsrf', 'wncsru', 'wncsfr', 'wncsfu', 'wncsur', 'wncsuf', 'wncurf', 'wncurs', 'wncufr', 'wncufs', 'wncusr', 'wncusf', 'wnfrcs', 'wnfrcu', 'wnfrsc', 'wnfrsu', 'wnfruc', 'wnfrus', 'wnfcrs', 'wnfcru', 'wnfcsr', 'wnfcsu', 'wnfcur', 'wnfcus', 'wnfsrc', 'wnfsru', 'wnfscr', 'wnfscu', 'wnfsur', 'wnfsuc', 'wnfurc', 'wnfurs', 'wnfucr', 'wnfucs', 'wnfusr', 'wnfusc', 'wnsrcf', 'wnsrcu', 'wnsrfc', 'wnsrfu', 'wnsruc', 'wnsruf', 'wnscrf', 'wnscru', 'wnscfr', 'wnscfu', 'wnscur', 'wnscuf', 'wnsfrc', 'wnsfru', 'wnsfcr', 'wnsfcu', 'wnsfur', 'wnsfuc', 'wnsurc', 'wnsurf', 'wnsucr', 'wnsucf', 'wnsufr', 'wnsufc', 'wnurcf', 'wnurcs', 'wnurfc', 'wnurfs', 'wnursc', 'wnursf', 'wnucrf', 'wnucrs', 'wnucfr', 'wnucfs', 'wnucsr', 'wnucsf', 'wnufrc', 'wnufrs', 'wnufcr', 'wnufcs', 'wnufsr', 'wnufsc', 'wnusrc', 'wnusrf', 'wnuscr', 'wnuscf', 'wnusfr', 'wnusfc', 'wfrcns', 'wfrcnu', 'wfrcsn', 'wfrcsu', 'wfrcun', 'wfrcus', 'wfrncs', 'wfrncu', 'wfrnsc', 'wfrnsu', 'wfrnuc', 'wfrnus', 'wfrscn', 'wfrscu', 'wfrsnc', 'wfrsnu', 'wfrsuc', 'wfrsun', 'wfrucn', 'wfrucs', 'wfrunc', 'wfruns', 'wfrusc', 'wfrusn', 'wfcrns', 'wfcrnu', 'wfcrsn', 'wfcrsu', 'wfcrun', 'wfcrus', 'wfcnrs', 'wfcnru', 'wfcnsr', 'wfcnsu', 'wfcnur', 'wfcnus', 'wfcsrn', 'wfcsru', 'wfcsnr', 'wfcsnu', 'wfcsur', 'wfcsun', 'wfcurn', 'wfcurs', 'wfcunr', 'wfcuns', 'wfcusr', 'wfcusn', 'wfnrcs', 'wfnrcu', 'wfnrsc', 'wfnrsu', 'wfnruc', 'wfnrus', 'wfncrs', 'wfncru', 'wfncsr', 'wfncsu', 'wfncur', 'wfncus', 'wfnsrc', 'wfnsru', 'wfnscr', 'wfnscu', 'wfnsur', 'wfnsuc', 'wfnurc', 'wfnurs', 'wfnucr', 'wfnucs', 'wfnusr', 'wfnusc', 'wfsrcn', 'wfsrcu', 'wfsrnc', 'wfsrnu', 'wfsruc', 'wfsrun', 'wfscrn', 'wfscru', 'wfscnr', 'wfscnu', 'wfscur', 'wfscun', 'wfsnrc', 'wfsnru', 'wfsncr', 'wfsncu', 'wfsnur', 'wfsnuc', 'wfsurc', 'wfsurn', 'wfsucr', 'wfsucn', 'wfsunr', 'wfsunc', 'wfurcn', 'wfurcs', 'wfurnc', 'wfurns', 'wfursc', 'wfursn', 'wfucrn', 'wfucrs', 'wfucnr', 'wfucns', 'wfucsr', 'wfucsn', 'wfunrc', 'wfunrs', 'wfuncr', 'wfuncs', 'wfunsr', 'wfunsc', 'wfusrc', 'wfusrn', 'wfuscr', 'wfuscn', 'wfusnr', 'wfusnc', 'wsrcnf', 'wsrcnu', 'wsrcfn', 'wsrcfu', 'wsrcun', 'wsrcuf', 'wsrncf', 'wsrncu', 'wsrnfc', 'wsrnfu', 'wsrnuc', 'wsrnuf', 'wsrfcn', 'wsrfcu', 'wsrfnc', 'wsrfnu', 'wsrfuc', 'wsrfun', 'wsrucn', 'wsrucf', 'wsrunc', 'wsrunf', 'wsrufc', 'wsrufn', 'wscrnf', 'wscrnu', 'wscrfn', 'wscrfu', 'wscrun', 'wscruf', 'wscnrf', 'wscnru', 'wscnfr', 'wscnfu', 'wscnur', 'wscnuf', 'wscfrn', 'wscfru', 'wscfnr', 'wscfnu', 'wscfur', 'wscfun', 'wscurn', 'wscurf', 'wscunr', 'wscunf', 'wscufr', 'wscufn', 'wsnrcf', 'wsnrcu', 'wsnrfc', 'wsnrfu', 'wsnruc', 'wsnruf', 'wsncrf', 'wsncru', 'wsncfr', 'wsncfu', 'wsncur', 'wsncuf', 'wsnfrc', 'wsnfru', 'wsnfcr', 'wsnfcu', 'wsnfur', 'wsnfuc', 'wsnurc', 'wsnurf', 'wsnucr', 'wsnucf', 'wsnufr', 'wsnufc', 'wsfrcn', 'wsfrcu', 'wsfrnc', 'wsfrnu', 'wsfruc', 'wsfrun', 'wsfcrn', 'wsfcru', 'wsfcnr', 'wsfcnu', 'wsfcur', 'wsfcun', 'wsfnrc', 'wsfnru', 'wsfncr', 'wsfncu', 'wsfnur', 'wsfnuc', 'wsfurc', 'wsfurn', 'wsfucr', 'wsfucn', 'wsfunr', 'wsfunc', 'wsurcn', 'wsurcf', 'wsurnc', 'wsurnf', 'wsurfc', 'wsurfn', 'wsucrn', 'wsucrf', 'wsucnr', 'wsucnf', 'wsucfr', 'wsucfn', 'wsunrc', 'wsunrf', 'wsuncr', 'wsuncf', 'wsunfr', 'wsunfc', 'wsufrc', 'wsufrn', 'wsufcr', 'wsufcn', 'wsufnr', 'wsufnc', 'wurcnf', 'wurcns', 'wurcfn', 'wurcfs', 'wurcsn', 'wurcsf', 'wurncf', 'wurncs', 'wurnfc', 'wurnfs', 'wurnsc', 'wurnsf', 'wurfcn', 'wurfcs', 'wurfnc', 'wurfns', 'wurfsc', 'wurfsn', 'wurscn', 'wurscf', 'wursnc', 'wursnf', 'wursfc', 'wursfn', 'wucrnf', 'wucrns', 'wucrfn', 'wucrfs', 'wucrsn', 'wucrsf', 'wucnrf', 'wucnrs', 'wucnfr', 'wucnfs', 'wucnsr', 'wucnsf', 'wucfrn', 'wucfrs', 'wucfnr', 'wucfns', 'wucfsr', 'wucfsn', 'wucsrn', 'wucsrf', 'wucsnr', 'wucsnf', 'wucsfr', 'wucsfn', 'wunrcf', 'wunrcs', 'wunrfc', 'wunrfs', 'wunrsc', 'wunrsf', 'wuncrf', 'wuncrs', 'wuncfr', 'wuncfs', 'wuncsr', 'wuncsf', 'wunfrc', 'wunfrs', 'wunfcr', 'wunfcs', 'wunfsr', 'wunfsc', 'wunsrc', 'wunsrf', 'wunscr', 'wunscf', 'wunsfr', 'wunsfc', 'wufrcn', 'wufrcs', 'wufrnc', 'wufrns', 'wufrsc', 'wufrsn', 'wufcrn', 'wufcrs', 'wufcnr', 'wufcns', 'wufcsr', 'wufcsn', 'wufnrc', 'wufnrs', 'wufncr', 'wufncs', 'wufnsr', 'wufnsc', 'wufsrc', 'wufsrn', 'wufscr', 'wufscn', 'wufsnr', 'wufsnc', 'wusrcn', 'wusrcf', 'wusrnc', 'wusrnf', 'wusrfc', 'wusrfn', 'wuscrn', 'wuscrf', 'wuscnr', 'wuscnf', 'wuscfr', 'wuscfn', 'wusnrc', 'wusnrf', 'wusncr', 'wusncf', 'wusnfr', 'wusnfc', 'wusfrc', 'wusfrn', 'wusfcr', 'wusfcn', 'wusfnr', 'wusfnc', 'crwnfs', 'crwnfu', 'crwnsf', 'crwnsu', 'crwnuf', 'crwnus', 'crwfns', 'crwfnu', 'crwfsn', 'crwfsu', 'crwfun', 'crwfus', 'crwsnf', 'crwsnu', 'crwsfn', 'crwsfu', 'crwsun', 'crwsuf', 'crwunf', 'crwuns', 'crwufn', 'crwufs', 'crwusn', 'crwusf', 'crnwfs', 'crnwfu', 'crnwsf', 'crnwsu', 'crnwuf', 'crnwus', 'crnfws', 'crnfwu', 'crnfsw', 'crnfsu', 'crnfuw', 'crnfus', 'crnswf', 'crnswu', 'crnsfw', 'crnsfu', 'crnsuw', 'crnsuf', 'crnuwf', 'crnuws', 'crnufw', 'crnufs', 'crnusw', 'crnusf', 'crfwns', 'crfwnu', 'crfwsn', 'crfwsu', 'crfwun', 'crfwus', 'crfnws', 'crfnwu', 'crfnsw', 'crfnsu', 'crfnuw', 'crfnus', 'crfswn', 'crfswu', 'crfsnw', 'crfsnu', 'crfsuw', 'crfsun', 'crfuwn', 'crfuws', 'crfunw', 'crfuns', 'crfusw', 'crfusn', 'crswnf', 'crswnu', 'crswfn', 'crswfu', 'crswun', 'crswuf', 'crsnwf', 'crsnwu', 'crsnfw', 'crsnfu', 'crsnuw', 'crsnuf', 'crsfwn', 'crsfwu', 'crsfnw', 'crsfnu', 'crsfuw', 'crsfun', 'crsuwn', 'crsuwf', 'crsunw', 'crsunf', 'crsufw', 'crsufn', 'cruwnf', 'cruwns', 'cruwfn', 'cruwfs', 'cruwsn', 'cruwsf', 'crunwf', 'crunws', 'crunfw', 'crunfs', 'crunsw', 'crunsf', 'crufwn', 'crufws', 'crufnw', 'crufns', 'crufsw', 'crufsn', 'cruswn', 'cruswf', 'crusnw', 'crusnf', 'crusfw', 'crusfn', 'cwrnfs', 'cwrnfu', 'cwrnsf', 'cwrnsu', 'cwrnuf', 'cwrnus', 'cwrfns', 'cwrfnu', 'cwrfsn', 'cwrfsu', 'cwrfun', 'cwrfus', 'cwrsnf', 'cwrsnu', 'cwrsfn', 'cwrsfu', 'cwrsun', 'cwrsuf', 'cwrunf', 'cwruns', 'cwrufn', 'cwrufs', 'cwrusn', 'cwrusf', 'cwnrfs', 'cwnrfu', 'cwnrsf', 'cwnrsu', 'cwnruf', 'cwnrus', 'cwnfrs', 'cwnfru', 'cwnfsr', 'cwnfsu', 'cwnfur', 'cwnfus', 'cwnsrf', 'cwnsru', 'cwnsfr', 'cwnsfu', 'cwnsur', 'cwnsuf', 'cwnurf', 'cwnurs', 'cwnufr', 'cwnufs', 'cwnusr', 'cwnusf', 'cwfrns', 'cwfrnu', 'cwfrsn', 'cwfrsu', 'cwfrun', 'cwfrus', 'cwfnrs', 'cwfnru', 'cwfnsr', 'cwfnsu', 'cwfnur', 'cwfnus', 'cwfsrn', 'cwfsru', 'cwfsnr', 'cwfsnu', 'cwfsur', 'cwfsun', 'cwfurn', 'cwfurs', 'cwfunr', 'cwfuns', 'cwfusr', 'cwfusn', 'cwsrnf', 'cwsrnu', 'cwsrfn', 'cwsrfu', 'cwsrun', 'cwsruf', 'cwsnrf', 'cwsnru', 'cwsnfr', 'cwsnfu', 'cwsnur', 'cwsnuf', 'cwsfrn', 'cwsfru', 'cwsfnr', 'cwsfnu', 'cwsfur', 'cwsfun', 'cwsurn', 'cwsurf', 'cwsunr', 'cwsunf', 'cwsufr', 'cwsufn', 'cwurnf', 'cwurns', 'cwurfn', 'cwurfs', 'cwursn', 'cwursf', 'cwunrf', 'cwunrs', 'cwunfr', 'cwunfs', 'cwunsr', 'cwunsf', 'cwufrn', 'cwufrs', 'cwufnr', 'cwufns', 'cwufsr', 'cwufsn', 'cwusrn', 'cwusrf', 'cwusnr', 'cwusnf', 'cwusfr', 'cwusfn', 'cnrwfs', 'cnrwfu', 'cnrwsf', 'cnrwsu', 'cnrwuf', 'cnrwus', 'cnrfws', 'cnrfwu', 'cnrfsw', 'cnrfsu', 'cnrfuw', 'cnrfus', 'cnrswf', 'cnrswu', 'cnrsfw', 'cnrsfu', 'cnrsuw', 'cnrsuf', 'cnruwf', 'cnruws', 'cnrufw', 'cnrufs', 'cnrusw', 'cnrusf', 'cnwrfs', 'cnwrfu', 'cnwrsf', 'cnwrsu', 'cnwruf', 'cnwrus', 'cnwfrs', 'cnwfru', 'cnwfsr', 'cnwfsu', 'cnwfur', 'cnwfus', 'cnwsrf', 'cnwsru', 'cnwsfr', 'cnwsfu', 'cnwsur', 'cnwsuf', 'cnwurf', 'cnwurs', 'cnwufr', 'cnwufs', 'cnwusr', 'cnwusf', 'cnfrws', 'cnfrwu', 'cnfrsw', 'cnfrsu', 'cnfruw', 'cnfrus', 'cnfwrs', 'cnfwru', 'cnfwsr', 'cnfwsu', 'cnfwur', 'cnfwus', 'cnfsrw', 'cnfsru', 'cnfswr', 'cnfswu', 'cnfsur', 'cnfsuw', 'cnfurw', 'cnfurs', 'cnfuwr', 'cnfuws', 'cnfusr', 'cnfusw', 'cnsrwf', 'cnsrwu', 'cnsrfw', 'cnsrfu', 'cnsruw', 'cnsruf', 'cnswrf', 'cnswru', 'cnswfr', 'cnswfu', 'cnswur', 'cnswuf', 'cnsfrw', 'cnsfru', 'cnsfwr', 'cnsfwu', 'cnsfur', 'cnsfuw', 'cnsurw', 'cnsurf', 'cnsuwr', 'cnsuwf', 'cnsufr', 'cnsufw', 'cnurwf', 'cnurws', 'cnurfw', 'cnurfs', 'cnursw', 'cnursf', 'cnuwrf', 'cnuwrs', 'cnuwfr', 'cnuwfs', 'cnuwsr', 'cnuwsf', 'cnufrw', 'cnufrs', 'cnufwr', 'cnufws', 'cnufsr', 'cnufsw', 'cnusrw', 'cnusrf', 'cnuswr', 'cnuswf', 'cnusfr', 'cnusfw', 'cfrwns', 'cfrwnu', 'cfrwsn', 'cfrwsu', 'cfrwun', 'cfrwus', 'cfrnws', 'cfrnwu', 'cfrnsw', 'cfrnsu', 'cfrnuw', 'cfrnus', 'cfrswn', 'cfrswu', 'cfrsnw', 'cfrsnu', 'cfrsuw', 'cfrsun', 'cfruwn', 'cfruws', 'cfrunw', 'cfruns', 'cfrusw', 'cfrusn', 'cfwrns', 'cfwrnu', 'cfwrsn', 'cfwrsu', 'cfwrun', 'cfwrus', 'cfwnrs', 'cfwnru', 'cfwnsr', 'cfwnsu', 'cfwnur', 'cfwnus', 'cfwsrn', 'cfwsru', 'cfwsnr', 'cfwsnu', 'cfwsur', 'cfwsun', 'cfwurn', 'cfwurs', 'cfwunr', 'cfwuns', 'cfwusr', 'cfwusn', 'cfnrws', 'cfnrwu', 'cfnrsw', 'cfnrsu', 'cfnruw', 'cfnrus', 'cfnwrs', 'cfnwru', 'cfnwsr', 'cfnwsu', 'cfnwur', 'cfnwus', 'cfnsrw', 'cfnsru', 'cfnswr', 'cfnswu', 'cfnsur', 'cfnsuw', 'cfnurw', 'cfnurs', 'cfnuwr', 'cfnuws', 'cfnusr', 'cfnusw', 'cfsrwn', 'cfsrwu', 'cfsrnw', 'cfsrnu', 'cfsruw', 'cfsrun', 'cfswrn', 'cfswru', 'cfswnr', 'cfswnu', 'cfswur', 'cfswun', 'cfsnrw', 'cfsnru', 'cfsnwr', 'cfsnwu', 'cfsnur', 'cfsnuw', 'cfsurw', 'cfsurn', 'cfsuwr', 'cfsuwn', 'cfsunr', 'cfsunw', 'cfurwn', 'cfurws', 'cfurnw', 'cfurns', 'cfursw', 'cfursn', 'cfuwrn', 'cfuwrs', 'cfuwnr', 'cfuwns', 'cfuwsr', 'cfuwsn', 'cfunrw', 'cfunrs', 'cfunwr', 'cfunws', 'cfunsr', 'cfunsw', 'cfusrw', 'cfusrn', 'cfuswr', 'cfuswn', 'cfusnr', 'cfusnw', 'csrwnf', 'csrwnu', 'csrwfn', 'csrwfu', 'csrwun', 'csrwuf', 'csrnwf', 'csrnwu', 'csrnfw', 'csrnfu', 'csrnuw', 'csrnuf', 'csrfwn', 'csrfwu', 'csrfnw', 'csrfnu', 'csrfuw', 'csrfun', 'csruwn', 'csruwf', 'csrunw', 'csrunf', 'csrufw', 'csrufn', 'cswrnf', 'cswrnu', 'cswrfn', 'cswrfu', 'cswrun', 'cswruf', 'cswnrf', 'cswnru', 'cswnfr', 'cswnfu', 'cswnur', 'cswnuf', 'cswfrn', 'cswfru', 'cswfnr', 'cswfnu', 'cswfur', 'cswfun', 'cswurn', 'cswurf', 'cswunr', 'cswunf', 'cswufr', 'cswufn', 'csnrwf', 'csnrwu', 'csnrfw', 'csnrfu', 'csnruw', 'csnruf', 'csnwrf', 'csnwru', 'csnwfr', 'csnwfu', 'csnwur', 'csnwuf', 'csnfrw', 'csnfru', 'csnfwr', 'csnfwu', 'csnfur', 'csnfuw', 'csnurw', 'csnurf', 'csnuwr', 'csnuwf', 'csnufr', 'csnufw', 'csfrwn', 'csfrwu', 'csfrnw', 'csfrnu', 'csfruw', 'csfrun', 'csfwrn', 'csfwru', 'csfwnr', 'csfwnu', 'csfwur', 'csfwun', 'csfnrw', 'csfnru', 'csfnwr', 'csfnwu', 'csfnur', 'csfnuw', 'csfurw', 'csfurn', 'csfuwr', 'csfuwn', 'csfunr', 'csfunw', 'csurwn', 'csurwf', 'csurnw', 'csurnf', 'csurfw', 'csurfn', 'csuwrn', 'csuwrf', 'csuwnr', 'csuwnf', 'csuwfr', 'csuwfn', 'csunrw', 'csunrf', 'csunwr', 'csunwf', 'csunfr', 'csunfw', 'csufrw', 'csufrn', 'csufwr', 'csufwn', 'csufnr', 'csufnw', 'curwnf', 'curwns', 'curwfn', 'curwfs', 'curwsn', 'curwsf', 'curnwf', 'curnws', 'curnfw', 'curnfs', 'curnsw', 'curnsf', 'curfwn', 'curfws', 'curfnw', 'curfns', 'curfsw', 'curfsn', 'curswn', 'curswf', 'cursnw', 'cursnf', 'cursfw', 'cursfn', 'cuwrnf', 'cuwrns', 'cuwrfn', 'cuwrfs', 'cuwrsn', 'cuwrsf', 'cuwnrf', 'cuwnrs', 'cuwnfr', 'cuwnfs', 'cuwnsr', 'cuwnsf', 'cuwfrn', 'cuwfrs', 'cuwfnr', 'cuwfns', 'cuwfsr', 'cuwfsn', 'cuwsrn', 'cuwsrf', 'cuwsnr', 'cuwsnf', 'cuwsfr', 'cuwsfn', 'cunrwf', 'cunrws', 'cunrfw', 'cunrfs', 'cunrsw', 'cunrsf', 'cunwrf', 'cunwrs', 'cunwfr', 'cunwfs', 'cunwsr', 'cunwsf', 'cunfrw', 'cunfrs', 'cunfwr', 'cunfws', 'cunfsr', 'cunfsw', 'cunsrw', 'cunsrf', 'cunswr', 'cunswf', 'cunsfr', 'cunsfw', 'cufrwn', 'cufrws', 'cufrnw', 'cufrns', 'cufrsw', 'cufrsn', 'cufwrn', 'cufwrs', 'cufwnr', 'cufwns', 'cufwsr', 'cufwsn', 'cufnrw', 'cufnrs', 'cufnwr', 'cufnws', 'cufnsr', 'cufnsw', 'cufsrw', 'cufsrn', 'cufswr', 'cufswn', 'cufsnr', 'cufsnw', 'cusrwn', 'cusrwf', 'cusrnw', 'cusrnf', 'cusrfw', 'cusrfn', 'cuswrn', 'cuswrf', 'cuswnr', 'cuswnf', 'cuswfr', 'cuswfn', 'cusnrw', 'cusnrf', 'cusnwr', 'cusnwf', 'cusnfr', 'cusnfw', 'cusfrw', 'cusfrn', 'cusfwr', 'cusfwn', 'cusfnr', 'cusfnw', 'nrwcfs', 'nrwcfu', 'nrwcsf', 'nrwcsu', 'nrwcuf', 'nrwcus', 'nrwfcs', 'nrwfcu', 'nrwfsc', 'nrwfsu', 'nrwfuc', 'nrwfus', 'nrwscf', 'nrwscu', 'nrwsfc', 'nrwsfu', 'nrwsuc', 'nrwsuf', 'nrwucf', 'nrwucs', 'nrwufc', 'nrwufs', 'nrwusc', 'nrwusf', 'nrcwfs', 'nrcwfu', 'nrcwsf', 'nrcwsu', 'nrcwuf', 'nrcwus', 'nrcfws', 'nrcfwu', 'nrcfsw', 'nrcfsu', 'nrcfuw', 'nrcfus', 'nrcswf', 'nrcswu', 'nrcsfw', 'nrcsfu', 'nrcsuw', 'nrcsuf', 'nrcuwf', 'nrcuws', 'nrcufw', 'nrcufs', 'nrcusw', 'nrcusf', 'nrfwcs', 'nrfwcu', 'nrfwsc', 'nrfwsu', 'nrfwuc', 'nrfwus', 'nrfcws', 'nrfcwu', 'nrfcsw', 'nrfcsu', 'nrfcuw', 'nrfcus', 'nrfswc', 'nrfswu', 'nrfscw', 'nrfscu', 'nrfsuw', 'nrfsuc', 'nrfuwc', 'nrfuws', 'nrfucw', 'nrfucs', 'nrfusw', 'nrfusc', 'nrswcf', 'nrswcu', 'nrswfc', 'nrswfu', 'nrswuc', 'nrswuf', 'nrscwf', 'nrscwu', 'nrscfw', 'nrscfu', 'nrscuw', 'nrscuf', 'nrsfwc', 'nrsfwu', 'nrsfcw', 'nrsfcu', 'nrsfuw', 'nrsfuc', 'nrsuwc', 'nrsuwf', 'nrsucw', 'nrsucf', 'nrsufw', 'nrsufc', 'nruwcf', 'nruwcs', 'nruwfc', 'nruwfs', 'nruwsc', 'nruwsf', 'nrucwf', 'nrucws', 'nrucfw', 'nrucfs', 'nrucsw', 'nrucsf', 'nrufwc', 'nrufws', 'nrufcw', 'nrufcs', 'nrufsw', 'nrufsc', 'nruswc', 'nruswf', 'nruscw', 'nruscf', 'nrusfw', 'nrusfc', 'nwrcfs', 'nwrcfu', 'nwrcsf', 'nwrcsu', 'nwrcuf', 'nwrcus', 'nwrfcs', 'nwrfcu', 'nwrfsc', 'nwrfsu', 'nwrfuc', 'nwrfus', 'nwrscf', 'nwrscu', 'nwrsfc', 'nwrsfu', 'nwrsuc', 'nwrsuf', 'nwrucf', 'nwrucs', 'nwrufc', 'nwrufs', 'nwrusc', 'nwrusf', 'nwcrfs', 'nwcrfu', 'nwcrsf', 'nwcrsu', 'nwcruf', 'nwcrus', 'nwcfrs', 'nwcfru', 'nwcfsr', 'nwcfsu', 'nwcfur', 'nwcfus', 'nwcsrf', 'nwcsru', 'nwcsfr', 'nwcsfu', 'nwcsur', 'nwcsuf', 'nwcurf', 'nwcurs', 'nwcufr', 'nwcufs', 'nwcusr', 'nwcusf', 'nwfrcs', 'nwfrcu', 'nwfrsc', 'nwfrsu', 'nwfruc', 'nwfrus', 'nwfcrs', 'nwfcru', 'nwfcsr', 'nwfcsu', 'nwfcur', 'nwfcus', 'nwfsrc', 'nwfsru', 'nwfscr', 'nwfscu', 'nwfsur', 'nwfsuc', 'nwfurc', 'nwfurs', 'nwfucr', 'nwfucs', 'nwfusr', 'nwfusc', 'nwsrcf', 'nwsrcu', 'nwsrfc', 'nwsrfu', 'nwsruc', 'nwsruf', 'nwscrf', 'nwscru', 'nwscfr', 'nwscfu', 'nwscur', 'nwscuf', 'nwsfrc', 'nwsfru', 'nwsfcr', 'nwsfcu', 'nwsfur', 'nwsfuc', 'nwsurc', 'nwsurf', 'nwsucr', 'nwsucf', 'nwsufr', 'nwsufc', 'nwurcf', 'nwurcs', 'nwurfc', 'nwurfs', 'nwursc', 'nwursf', 'nwucrf', 'nwucrs', 'nwucfr', 'nwucfs', 'nwucsr', 'nwucsf', 'nwufrc', 'nwufrs', 'nwufcr', 'nwufcs', 'nwufsr', 'nwufsc', 'nwusrc', 'nwusrf', 'nwuscr', 'nwuscf', 'nwusfr', 'nwusfc', 'ncrwfs', 'ncrwfu', 'ncrwsf', 'ncrwsu', 'ncrwuf', 'ncrwus', 'ncrfws', 'ncrfwu', 'ncrfsw', 'ncrfsu', 'ncrfuw', 'ncrfus', 'ncrswf', 'ncrswu', 'ncrsfw', 'ncrsfu', 'ncrsuw', 'ncrsuf', 'ncruwf', 'ncruws', 'ncrufw', 'ncrufs', 'ncrusw', 'ncrusf', 'ncwrfs', 'ncwrfu', 'ncwrsf', 'ncwrsu', 'ncwruf', 'ncwrus', 'ncwfrs', 'ncwfru', 'ncwfsr', 'ncwfsu', 'ncwfur', 'ncwfus', 'ncwsrf', 'ncwsru', 'ncwsfr', 'ncwsfu', 'ncwsur', 'ncwsuf', 'ncwurf', 'ncwurs', 'ncwufr', 'ncwufs', 'ncwusr', 'ncwusf', 'ncfrws', 'ncfrwu', 'ncfrsw', 'ncfrsu', 'ncfruw', 'ncfrus', 'ncfwrs', 'ncfwru', 'ncfwsr', 'ncfwsu', 'ncfwur', 'ncfwus', 'ncfsrw', 'ncfsru', 'ncfswr', 'ncfswu', 'ncfsur', 'ncfsuw', 'ncfurw', 'ncfurs', 'ncfuwr', 'ncfuws', 'ncfusr', 'ncfusw', 'ncsrwf', 'ncsrwu', 'ncsrfw', 'ncsrfu', 'ncsruw', 'ncsruf', 'ncswrf', 'ncswru', 'ncswfr', 'ncswfu', 'ncswur', 'ncswuf', 'ncsfrw', 'ncsfru', 'ncsfwr', 'ncsfwu', 'ncsfur', 'ncsfuw', 'ncsurw', 'ncsurf', 'ncsuwr', 'ncsuwf', 'ncsufr', 'ncsufw', 'ncurwf', 'ncurws', 'ncurfw', 'ncurfs', 'ncursw', 'ncursf', 'ncuwrf', 'ncuwrs', 'ncuwfr', 'ncuwfs', 'ncuwsr', 'ncuwsf', 'ncufrw', 'ncufrs', 'ncufwr', 'ncufws', 'ncufsr', 'ncufsw', 'ncusrw', 'ncusrf', 'ncuswr', 'ncuswf', 'ncusfr', 'ncusfw', 'nfrwcs', 'nfrwcu', 'nfrwsc', 'nfrwsu', 'nfrwuc', 'nfrwus', 'nfrcws', 'nfrcwu', 'nfrcsw', 'nfrcsu', 'nfrcuw', 'nfrcus', 'nfrswc', 'nfrswu', 'nfrscw', 'nfrscu', 'nfrsuw', 'nfrsuc', 'nfruwc', 'nfruws', 'nfrucw', 'nfrucs', 'nfrusw', 'nfrusc', 'nfwrcs', 'nfwrcu', 'nfwrsc', 'nfwrsu', 'nfwruc', 'nfwrus', 'nfwcrs', 'nfwcru', 'nfwcsr', 'nfwcsu', 'nfwcur', 'nfwcus', 'nfwsrc', 'nfwsru', 'nfwscr', 'nfwscu', 'nfwsur', 'nfwsuc', 'nfwurc', 'nfwurs', 'nfwucr', 'nfwucs', 'nfwusr', 'nfwusc', 'nfcrws', 'nfcrwu', 'nfcrsw', 'nfcrsu', 'nfcruw', 'nfcrus', 'nfcwrs', 'nfcwru', 'nfcwsr', 'nfcwsu', 'nfcwur', 'nfcwus', 'nfcsrw', 'nfcsru', 'nfcswr', 'nfcswu', 'nfcsur', 'nfcsuw', 'nfcurw', 'nfcurs', 'nfcuwr', 'nfcuws', 'nfcusr', 'nfcusw', 'nfsrwc', 'nfsrwu', 'nfsrcw', 'nfsrcu', 'nfsruw', 'nfsruc', 'nfswrc', 'nfswru', 'nfswcr', 'nfswcu', 'nfswur', 'nfswuc', 'nfscrw', 'nfscru', 'nfscwr', 'nfscwu', 'nfscur', 'nfscuw', 'nfsurw', 'nfsurc', 'nfsuwr', 'nfsuwc', 'nfsucr', 'nfsucw', 'nfurwc', 'nfurws', 'nfurcw', 'nfurcs', 'nfursw', 'nfursc', 'nfuwrc', 'nfuwrs', 'nfuwcr', 'nfuwcs', 'nfuwsr', 'nfuwsc', 'nfucrw', 'nfucrs', 'nfucwr', 'nfucws', 'nfucsr', 'nfucsw', 'nfusrw', 'nfusrc', 'nfuswr', 'nfuswc', 'nfuscr', 'nfuscw', 'nsrwcf', 'nsrwcu', 'nsrwfc', 'nsrwfu', 'nsrwuc', 'nsrwuf', 'nsrcwf', 'nsrcwu', 'nsrcfw', 'nsrcfu', 'nsrcuw', 'nsrcuf', 'nsrfwc', 'nsrfwu', 'nsrfcw', 'nsrfcu', 'nsrfuw', 'nsrfuc', 'nsruwc', 'nsruwf', 'nsrucw', 'nsrucf', 'nsrufw', 'nsrufc', 'nswrcf', 'nswrcu', 'nswrfc', 'nswrfu', 'nswruc', 'nswruf', 'nswcrf', 'nswcru', 'nswcfr', 'nswcfu', 'nswcur', 'nswcuf', 'nswfrc', 'nswfru', 'nswfcr', 'nswfcu', 'nswfur', 'nswfuc', 'nswurc', 'nswurf', 'nswucr', 'nswucf', 'nswufr', 'nswufc', 'nscrwf', 'nscrwu', 'nscrfw', 'nscrfu', 'nscruw', 'nscruf', 'nscwrf', 'nscwru', 'nscwfr', 'nscwfu', 'nscwur', 'nscwuf', 'nscfrw', 'nscfru', 'nscfwr', 'nscfwu', 'nscfur', 'nscfuw', 'nscurw', 'nscurf', 'nscuwr', 'nscuwf', 'nscufr', 'nscufw', 'nsfrwc', 'nsfrwu', 'nsfrcw', 'nsfrcu', 'nsfruw', 'nsfruc', 'nsfwrc', 'nsfwru', 'nsfwcr', 'nsfwcu', 'nsfwur', 'nsfwuc', 'nsfcrw', 'nsfcru', 'nsfcwr', 'nsfcwu', 'nsfcur', 'nsfcuw', 'nsfurw', 'nsfurc', 'nsfuwr', 'nsfuwc', 'nsfucr', 'nsfucw', 'nsurwc', 'nsurwf', 'nsurcw', 'nsurcf', 'nsurfw', 'nsurfc', 'nsuwrc', 'nsuwrf', 'nsuwcr', 'nsuwcf', 'nsuwfr', 'nsuwfc', 'nsucrw', 'nsucrf', 'nsucwr', 'nsucwf', 'nsucfr', 'nsucfw', 'nsufrw', 'nsufrc', 'nsufwr', 'nsufwc', 'nsufcr', 'nsufcw', 'nurwcf', 'nurwcs', 'nurwfc', 'nurwfs', 'nurwsc', 'nurwsf', 'nurcwf', 'nurcws', 'nurcfw', 'nurcfs', 'nurcsw', 'nurcsf', 'nurfwc', 'nurfws', 'nurfcw', 'nurfcs', 'nurfsw', 'nurfsc', 'nurswc', 'nurswf', 'nurscw', 'nurscf', 'nursfw', 'nursfc', 'nuwrcf', 'nuwrcs', 'nuwrfc', 'nuwrfs', 'nuwrsc', 'nuwrsf', 'nuwcrf', 'nuwcrs', 'nuwcfr', 'nuwcfs', 'nuwcsr', 'nuwcsf', 'nuwfrc', 'nuwfrs', 'nuwfcr', 'nuwfcs', 'nuwfsr', 'nuwfsc', 'nuwsrc', 'nuwsrf', 'nuwscr', 'nuwscf', 'nuwsfr', 'nuwsfc', 'nucrwf', 'nucrws', 'nucrfw', 'nucrfs', 'nucrsw', 'nucrsf', 'nucwrf', 'nucwrs', 'nucwfr', 'nucwfs', 'nucwsr', 'nucwsf', 'nucfrw', 'nucfrs', 'nucfwr', 'nucfws', 'nucfsr', 'nucfsw', 'nucsrw', 'nucsrf', 'nucswr', 'nucswf', 'nucsfr', 'nucsfw', 'nufrwc', 'nufrws', 'nufrcw', 'nufrcs', 'nufrsw', 'nufrsc', 'nufwrc', 'nufwrs', 'nufwcr', 'nufwcs', 'nufwsr', 'nufwsc', 'nufcrw', 'nufcrs', 'nufcwr', 'nufcws', 'nufcsr', 'nufcsw', 'nufsrw', 'nufsrc', 'nufswr', 'nufswc', 'nufscr', 'nufscw', 'nusrwc', 'nusrwf', 'nusrcw', 'nusrcf', 'nusrfw', 'nusrfc', 'nuswrc', 'nuswrf', 'nuswcr', 'nuswcf', 'nuswfr', 'nuswfc', 'nuscrw', 'nuscrf', 'nuscwr', 'nuscwf', 'nuscfr', 'nuscfw', 'nusfrw', 'nusfrc', 'nusfwr', 'nusfwc', 'nusfcr', 'nusfcw', 'frwcns', 'frwcnu', 'frwcsn', 'frwcsu', 'frwcun', 'frwcus', 'frwncs', 'frwncu', 'frwnsc', 'frwnsu', 'frwnuc', 'frwnus', 'frwscn', 'frwscu', 'frwsnc', 'frwsnu', 'frwsuc', 'frwsun', 'frwucn', 'frwucs', 'frwunc', 'frwuns', 'frwusc', 'frwusn', 'frcwns', 'frcwnu', 'frcwsn', 'frcwsu', 'frcwun', 'frcwus', 'frcnws', 'frcnwu', 'frcnsw', 'frcnsu', 'frcnuw', 'frcnus', 'frcswn', 'frcswu', 'frcsnw', 'frcsnu', 'frcsuw', 'frcsun', 'frcuwn', 'frcuws', 'frcunw', 'frcuns', 'frcusw', 'frcusn', 'frnwcs', 'frnwcu', 'frnwsc', 'frnwsu', 'frnwuc', 'frnwus', 'frncws', 'frncwu', 'frncsw', 'frncsu', 'frncuw', 'frncus', 'frnswc', 'frnswu', 'frnscw', 'frnscu', 'frnsuw', 'frnsuc', 'frnuwc', 'frnuws', 'frnucw', 'frnucs', 'frnusw', 'frnusc', 'frswcn', 'frswcu', 'frswnc', 'frswnu', 'frswuc', 'frswun', 'frscwn', 'frscwu', 'frscnw', 'frscnu', 'frscuw', 'frscun', 'frsnwc', 'frsnwu', 'frsncw', 'frsncu', 'frsnuw', 'frsnuc', 'frsuwc', 'frsuwn', 'frsucw', 'frsucn', 'frsunw', 'frsunc', 'fruwcn', 'fruwcs', 'fruwnc', 'fruwns', 'fruwsc', 'fruwsn', 'frucwn', 'frucws', 'frucnw', 'frucns', 'frucsw', 'frucsn', 'frunwc', 'frunws', 'fruncw', 'fruncs', 'frunsw', 'frunsc', 'fruswc', 'fruswn', 'fruscw', 'fruscn', 'frusnw', 'frusnc', 'fwrcns', 'fwrcnu', 'fwrcsn', 'fwrcsu', 'fwrcun', 'fwrcus', 'fwrncs', 'fwrncu', 'fwrnsc', 'fwrnsu', 'fwrnuc', 'fwrnus', 'fwrscn', 'fwrscu', 'fwrsnc', 'fwrsnu', 'fwrsuc', 'fwrsun', 'fwrucn', 'fwrucs', 'fwrunc', 'fwruns', 'fwrusc', 'fwrusn', 'fwcrns', 'fwcrnu', 'fwcrsn', 'fwcrsu', 'fwcrun', 'fwcrus', 'fwcnrs', 'fwcnru', 'fwcnsr', 'fwcnsu', 'fwcnur', 'fwcnus', 'fwcsrn', 'fwcsru', 'fwcsnr', 'fwcsnu', 'fwcsur', 'fwcsun', 'fwcurn', 'fwcurs', 'fwcunr', 'fwcuns', 'fwcusr', 'fwcusn', 'fwnrcs', 'fwnrcu', 'fwnrsc', 'fwnrsu', 'fwnruc', 'fwnrus', 'fwncrs', 'fwncru', 'fwncsr', 'fwncsu', 'fwncur', 'fwncus', 'fwnsrc', 'fwnsru', 'fwnscr', 'fwnscu', 'fwnsur', 'fwnsuc', 'fwnurc', 'fwnurs', 'fwnucr', 'fwnucs', 'fwnusr', 'fwnusc', 'fwsrcn', 'fwsrcu', 'fwsrnc', 'fwsrnu', 'fwsruc', 'fwsrun', 'fwscrn', 'fwscru', 'fwscnr', 'fwscnu', 'fwscur', 'fwscun', 'fwsnrc', 'fwsnru', 'fwsncr', 'fwsncu', 'fwsnur', 'fwsnuc', 'fwsurc', 'fwsurn', 'fwsucr', 'fwsucn', 'fwsunr', 'fwsunc', 'fwurcn', 'fwurcs', 'fwurnc', 'fwurns', 'fwursc', 'fwursn', 'fwucrn', 'fwucrs', 'fwucnr', 'fwucns', 'fwucsr', 'fwucsn', 'fwunrc', 'fwunrs', 'fwuncr', 'fwuncs', 'fwunsr', 'fwunsc', 'fwusrc', 'fwusrn', 'fwuscr', 'fwuscn', 'fwusnr', 'fwusnc', 'fcrwns', 'fcrwnu', 'fcrwsn', 'fcrwsu', 'fcrwun', 'fcrwus', 'fcrnws', 'fcrnwu', 'fcrnsw', 'fcrnsu', 'fcrnuw', 'fcrnus', 'fcrswn', 'fcrswu', 'fcrsnw', 'fcrsnu', 'fcrsuw', 'fcrsun', 'fcruwn', 'fcruws', 'fcrunw', 'fcruns', 'fcrusw', 'fcrusn', 'fcwrns', 'fcwrnu', 'fcwrsn', 'fcwrsu', 'fcwrun', 'fcwrus', 'fcwnrs', 'fcwnru', 'fcwnsr', 'fcwnsu', 'fcwnur', 'fcwnus', 'fcwsrn', 'fcwsru', 'fcwsnr', 'fcwsnu', 'fcwsur', 'fcwsun', 'fcwurn', 'fcwurs', 'fcwunr', 'fcwuns', 'fcwusr', 'fcwusn', 'fcnrws', 'fcnrwu', 'fcnrsw', 'fcnrsu', 'fcnruw', 'fcnrus', 'fcnwrs', 'fcnwru', 'fcnwsr', 'fcnwsu', 'fcnwur', 'fcnwus', 'fcnsrw', 'fcnsru', 'fcnswr', 'fcnswu', 'fcnsur', 'fcnsuw', 'fcnurw', 'fcnurs', 'fcnuwr', 'fcnuws', 'fcnusr', 'fcnusw', 'fcsrwn', 'fcsrwu', 'fcsrnw', 'fcsrnu', 'fcsruw', 'fcsrun', 'fcswrn', 'fcswru', 'fcswnr', 'fcswnu', 'fcswur', 'fcswun', 'fcsnrw', 'fcsnru', 'fcsnwr', 'fcsnwu', 'fcsnur', 'fcsnuw', 'fcsurw', 'fcsurn', 'fcsuwr', 'fcsuwn', 'fcsunr', 'fcsunw', 'fcurwn', 'fcurws', 'fcurnw', 'fcurns', 'fcursw', 'fcursn', 'fcuwrn', 'fcuwrs', 'fcuwnr', 'fcuwns', 'fcuwsr', 'fcuwsn', 'fcunrw', 'fcunrs', 'fcunwr', 'fcunws', 'fcunsr', 'fcunsw', 'fcusrw', 'fcusrn', 'fcuswr', 'fcuswn', 'fcusnr', 'fcusnw', 'fnrwcs', 'fnrwcu', 'fnrwsc', 'fnrwsu', 'fnrwuc', 'fnrwus', 'fnrcws', 'fnrcwu', 'fnrcsw', 'fnrcsu', 'fnrcuw', 'fnrcus', 'fnrswc', 'fnrswu', 'fnrscw', 'fnrscu', 'fnrsuw', 'fnrsuc', 'fnruwc', 'fnruws', 'fnrucw', 'fnrucs', 'fnrusw', 'fnrusc', 'fnwrcs', 'fnwrcu', 'fnwrsc', 'fnwrsu', 'fnwruc', 'fnwrus', 'fnwcrs', 'fnwcru', 'fnwcsr', 'fnwcsu', 'fnwcur', 'fnwcus', 'fnwsrc', 'fnwsru', 'fnwscr', 'fnwscu', 'fnwsur', 'fnwsuc', 'fnwurc', 'fnwurs', 'fnwucr', 'fnwucs', 'fnwusr', 'fnwusc', 'fncrws', 'fncrwu', 'fncrsw', 'fncrsu', 'fncruw', 'fncrus', 'fncwrs', 'fncwru', 'fncwsr', 'fncwsu', 'fncwur', 'fncwus', 'fncsrw', 'fncsru', 'fncswr', 'fncswu', 'fncsur', 'fncsuw', 'fncurw', 'fncurs', 'fncuwr', 'fncuws', 'fncusr', 'fncusw', 'fnsrwc', 'fnsrwu', 'fnsrcw', 'fnsrcu', 'fnsruw', 'fnsruc', 'fnswrc', 'fnswru', 'fnswcr', 'fnswcu', 'fnswur', 'fnswuc', 'fnscrw', 'fnscru', 'fnscwr', 'fnscwu', 'fnscur', 'fnscuw', 'fnsurw', 'fnsurc', 'fnsuwr', 'fnsuwc', 'fnsucr', 'fnsucw', 'fnurwc', 'fnurws', 'fnurcw', 'fnurcs', 'fnursw', 'fnursc', 'fnuwrc', 'fnuwrs', 'fnuwcr', 'fnuwcs', 'fnuwsr', 'fnuwsc', 'fnucrw', 'fnucrs', 'fnucwr', 'fnucws', 'fnucsr', 'fnucsw', 'fnusrw', 'fnusrc', 'fnuswr', 'fnuswc', 'fnuscr', 'fnuscw', 'fsrwcn', 'fsrwcu', 'fsrwnc', 'fsrwnu', 'fsrwuc', 'fsrwun', 'fsrcwn', 'fsrcwu', 'fsrcnw', 'fsrcnu', 'fsrcuw', 'fsrcun', 'fsrnwc', 'fsrnwu', 'fsrncw', 'fsrncu', 'fsrnuw', 'fsrnuc', 'fsruwc', 'fsruwn', 'fsrucw', 'fsrucn', 'fsrunw', 'fsrunc', 'fswrcn', 'fswrcu', 'fswrnc', 'fswrnu', 'fswruc', 'fswrun', 'fswcrn', 'fswcru', 'fswcnr', 'fswcnu', 'fswcur', 'fswcun', 'fswnrc', 'fswnru', 'fswncr', 'fswncu', 'fswnur', 'fswnuc', 'fswurc', 'fswurn', 'fswucr', 'fswucn', 'fswunr', 'fswunc', 'fscrwn', 'fscrwu', 'fscrnw', 'fscrnu', 'fscruw', 'fscrun', 'fscwrn', 'fscwru', 'fscwnr', 'fscwnu', 'fscwur', 'fscwun', 'fscnrw', 'fscnru', 'fscnwr', 'fscnwu', 'fscnur', 'fscnuw', 'fscurw', 'fscurn', 'fscuwr', 'fscuwn', 'fscunr', 'fscunw', 'fsnrwc', 'fsnrwu', 'fsnrcw', 'fsnrcu', 'fsnruw', 'fsnruc', 'fsnwrc', 'fsnwru', 'fsnwcr', 'fsnwcu', 'fsnwur', 'fsnwuc', 'fsncrw', 'fsncru', 'fsncwr', 'fsncwu', 'fsncur', 'fsncuw', 'fsnurw', 'fsnurc', 'fsnuwr', 'fsnuwc', 'fsnucr', 'fsnucw', 'fsurwc', 'fsurwn', 'fsurcw', 'fsurcn', 'fsurnw', 'fsurnc', 'fsuwrc', 'fsuwrn', 'fsuwcr', 'fsuwcn', 'fsuwnr', 'fsuwnc', 'fsucrw', 'fsucrn', 'fsucwr', 'fsucwn', 'fsucnr', 'fsucnw', 'fsunrw', 'fsunrc', 'fsunwr', 'fsunwc', 'fsuncr', 'fsuncw', 'furwcn', 'furwcs', 'furwnc', 'furwns', 'furwsc', 'furwsn', 'furcwn', 'furcws', 'furcnw', 'furcns', 'furcsw', 'furcsn', 'furnwc', 'furnws', 'furncw', 'furncs', 'furnsw', 'furnsc', 'furswc', 'furswn', 'furscw', 'furscn', 'fursnw', 'fursnc', 'fuwrcn', 'fuwrcs', 'fuwrnc', 'fuwrns', 'fuwrsc', 'fuwrsn', 'fuwcrn', 'fuwcrs', 'fuwcnr', 'fuwcns', 'fuwcsr', 'fuwcsn', 'fuwnrc', 'fuwnrs', 'fuwncr', 'fuwncs', 'fuwnsr', 'fuwnsc', 'fuwsrc', 'fuwsrn', 'fuwscr', 'fuwscn', 'fuwsnr', 'fuwsnc', 'fucrwn', 'fucrws', 'fucrnw', 'fucrns', 'fucrsw', 'fucrsn', 'fucwrn', 'fucwrs', 'fucwnr', 'fucwns', 'fucwsr', 'fucwsn', 'fucnrw', 'fucnrs', 'fucnwr', 'fucnws', 'fucnsr', 'fucnsw', 'fucsrw', 'fucsrn', 'fucswr', 'fucswn', 'fucsnr', 'fucsnw', 'funrwc', 'funrws', 'funrcw', 'funrcs', 'funrsw', 'funrsc', 'funwrc', 'funwrs', 'funwcr', 'funwcs', 'funwsr', 'funwsc', 'funcrw', 'funcrs', 'funcwr', 'funcws', 'funcsr', 'funcsw', 'funsrw', 'funsrc', 'funswr', 'funswc', 'funscr', 'funscw', 'fusrwc', 'fusrwn', 'fusrcw', 'fusrcn', 'fusrnw', 'fusrnc', 'fuswrc', 'fuswrn', 'fuswcr', 'fuswcn', 'fuswnr', 'fuswnc', 'fuscrw', 'fuscrn', 'fuscwr', 'fuscwn', 'fuscnr', 'fuscnw', 'fusnrw', 'fusnrc', 'fusnwr', 'fusnwc', 'fusncr', 'fusncw', 'srwcnf', 'srwcnu', 'srwcfn', 'srwcfu', 'srwcun', 'srwcuf', 'srwncf', 'srwncu', 'srwnfc', 'srwnfu', 'srwnuc', 'srwnuf', 'srwfcn', 'srwfcu', 'srwfnc', 'srwfnu', 'srwfuc', 'srwfun', 'srwucn', 'srwucf', 'srwunc', 'srwunf', 'srwufc', 'srwufn', 'srcwnf', 'srcwnu', 'srcwfn', 'srcwfu', 'srcwun', 'srcwuf', 'srcnwf', 'srcnwu', 'srcnfw', 'srcnfu', 'srcnuw', 'srcnuf', 'srcfwn', 'srcfwu', 'srcfnw', 'srcfnu', 'srcfuw', 'srcfun', 'srcuwn', 'srcuwf', 'srcunw', 'srcunf', 'srcufw', 'srcufn', 'srnwcf', 'srnwcu', 'srnwfc', 'srnwfu', 'srnwuc', 'srnwuf', 'srncwf', 'srncwu', 'srncfw', 'srncfu', 'srncuw', 'srncuf', 'srnfwc', 'srnfwu', 'srnfcw', 'srnfcu', 'srnfuw', 'srnfuc', 'srnuwc', 'srnuwf', 'srnucw', 'srnucf', 'srnufw', 'srnufc', 'srfwcn', 'srfwcu', 'srfwnc', 'srfwnu', 'srfwuc', 'srfwun', 'srfcwn', 'srfcwu', 'srfcnw', 'srfcnu', 'srfcuw', 'srfcun', 'srfnwc', 'srfnwu', 'srfncw', 'srfncu', 'srfnuw', 'srfnuc', 'srfuwc', 'srfuwn', 'srfucw', 'srfucn', 'srfunw', 'srfunc', 'sruwcn', 'sruwcf', 'sruwnc', 'sruwnf', 'sruwfc', 'sruwfn', 'srucwn', 'srucwf', 'srucnw', 'srucnf', 'srucfw', 'srucfn', 'srunwc', 'srunwf', 'sruncw', 'sruncf', 'srunfw', 'srunfc', 'srufwc', 'srufwn', 'srufcw', 'srufcn', 'srufnw', 'srufnc', 'swrcnf', 'swrcnu', 'swrcfn', 'swrcfu', 'swrcun', 'swrcuf', 'swrncf', 'swrncu', 'swrnfc', 'swrnfu', 'swrnuc', 'swrnuf', 'swrfcn', 'swrfcu', 'swrfnc', 'swrfnu', 'swrfuc', 'swrfun', 'swrucn', 'swrucf', 'swrunc', 'swrunf', 'swrufc', 'swrufn', 'swcrnf', 'swcrnu', 'swcrfn', 'swcrfu', 'swcrun', 'swcruf', 'swcnrf', 'swcnru', 'swcnfr', 'swcnfu', 'swcnur', 'swcnuf', 'swcfrn', 'swcfru', 'swcfnr', 'swcfnu', 'swcfur', 'swcfun', 'swcurn', 'swcurf', 'swcunr', 'swcunf', 'swcufr', 'swcufn', 'swnrcf', 'swnrcu', 'swnrfc', 'swnrfu', 'swnruc', 'swnruf', 'swncrf', 'swncru', 'swncfr', 'swncfu', 'swncur', 'swncuf', 'swnfrc', 'swnfru', 'swnfcr', 'swnfcu', 'swnfur', 'swnfuc', 'swnurc', 'swnurf', 'swnucr', 'swnucf', 'swnufr', 'swnufc', 'swfrcn', 'swfrcu', 'swfrnc', 'swfrnu', 'swfruc', 'swfrun', 'swfcrn', 'swfcru', 'swfcnr', 'swfcnu', 'swfcur', 'swfcun', 'swfnrc', 'swfnru', 'swfncr', 'swfncu', 'swfnur', 'swfnuc', 'swfurc', 'swfurn', 'swfucr', 'swfucn', 'swfunr', 'swfunc', 'swurcn', 'swurcf', 'swurnc', 'swurnf', 'swurfc', 'swurfn', 'swucrn', 'swucrf', 'swucnr', 'swucnf', 'swucfr', 'swucfn', 'swunrc', 'swunrf', 'swuncr', 'swuncf', 'swunfr', 'swunfc', 'swufrc', 'swufrn', 'swufcr', 'swufcn', 'swufnr', 'swufnc', 'scrwnf', 'scrwnu', 'scrwfn', 'scrwfu', 'scrwun', 'scrwuf', 'scrnwf', 'scrnwu', 'scrnfw', 'scrnfu', 'scrnuw', 'scrnuf', 'scrfwn', 'scrfwu', 'scrfnw', 'scrfnu', 'scrfuw', 'scrfun', 'scruwn', 'scruwf', 'scrunw', 'scrunf', 'scrufw', 'scrufn', 'scwrnf', 'scwrnu', 'scwrfn', 'scwrfu', 'scwrun', 'scwruf', 'scwnrf', 'scwnru', 'scwnfr', 'scwnfu', 'scwnur', 'scwnuf', 'scwfrn', 'scwfru', 'scwfnr', 'scwfnu', 'scwfur', 'scwfun', 'scwurn', 'scwurf', 'scwunr', 'scwunf', 'scwufr', 'scwufn', 'scnrwf', 'scnrwu', 'scnrfw', 'scnrfu', 'scnruw', 'scnruf', 'scnwrf', 'scnwru', 'scnwfr', 'scnwfu', 'scnwur', 'scnwuf', 'scnfrw', 'scnfru', 'scnfwr', 'scnfwu', 'scnfur', 'scnfuw', 'scnurw', 'scnurf', 'scnuwr', 'scnuwf', 'scnufr', 'scnufw', 'scfrwn', 'scfrwu', 'scfrnw', 'scfrnu', 'scfruw', 'scfrun', 'scfwrn', 'scfwru', 'scfwnr', 'scfwnu', 'scfwur', 'scfwun', 'scfnrw', 'scfnru', 'scfnwr', 'scfnwu', 'scfnur', 'scfnuw', 'scfurw', 'scfurn', 'scfuwr', 'scfuwn', 'scfunr', 'scfunw', 'scurwn', 'scurwf', 'scurnw', 'scurnf', 'scurfw', 'scurfn', 'scuwrn', 'scuwrf', 'scuwnr', 'scuwnf', 'scuwfr', 'scuwfn', 'scunrw', 'scunrf', 'scunwr', 'scunwf', 'scunfr', 'scunfw', 'scufrw', 'scufrn', 'scufwr', 'scufwn', 'scufnr', 'scufnw', 'snrwcf', 'snrwcu', 'snrwfc', 'snrwfu', 'snrwuc', 'snrwuf', 'snrcwf', 'snrcwu', 'snrcfw', 'snrcfu', 'snrcuw', 'snrcuf', 'snrfwc', 'snrfwu', 'snrfcw', 'snrfcu', 'snrfuw', 'snrfuc', 'snruwc', 'snruwf', 'snrucw', 'snrucf', 'snrufw', 'snrufc', 'snwrcf', 'snwrcu', 'snwrfc', 'snwrfu', 'snwruc', 'snwruf', 'snwcrf', 'snwcru', 'snwcfr', 'snwcfu', 'snwcur', 'snwcuf', 'snwfrc', 'snwfru', 'snwfcr', 'snwfcu', 'snwfur', 'snwfuc', 'snwurc', 'snwurf', 'snwucr', 'snwucf', 'snwufr', 'snwufc', 'sncrwf', 'sncrwu', 'sncrfw', 'sncrfu', 'sncruw', 'sncruf', 'sncwrf', 'sncwru', 'sncwfr', 'sncwfu', 'sncwur', 'sncwuf', 'sncfrw', 'sncfru', 'sncfwr', 'sncfwu', 'sncfur', 'sncfuw', 'sncurw', 'sncurf', 'sncuwr', 'sncuwf', 'sncufr', 'sncufw', 'snfrwc', 'snfrwu', 'snfrcw', 'snfrcu', 'snfruw', 'snfruc', 'snfwrc', 'snfwru', 'snfwcr', 'snfwcu', 'snfwur', 'snfwuc', 'snfcrw', 'snfcru', 'snfcwr', 'snfcwu', 'snfcur', 'snfcuw', 'snfurw', 'snfurc', 'snfuwr', 'snfuwc', 'snfucr', 'snfucw', 'snurwc', 'snurwf', 'snurcw', 'snurcf', 'snurfw', 'snurfc', 'snuwrc', 'snuwrf', 'snuwcr', 'snuwcf', 'snuwfr', 'snuwfc', 'snucrw', 'snucrf', 'snucwr', 'snucwf', 'snucfr', 'snucfw', 'snufrw', 'snufrc', 'snufwr', 'snufwc', 'snufcr', 'snufcw', 'sfrwcn', 'sfrwcu', 'sfrwnc', 'sfrwnu', 'sfrwuc', 'sfrwun', 'sfrcwn', 'sfrcwu', 'sfrcnw', 'sfrcnu', 'sfrcuw', 'sfrcun', 'sfrnwc', 'sfrnwu', 'sfrncw', 'sfrncu', 'sfrnuw', 'sfrnuc', 'sfruwc', 'sfruwn', 'sfrucw', 'sfrucn', 'sfrunw', 'sfrunc', 'sfwrcn', 'sfwrcu', 'sfwrnc', 'sfwrnu', 'sfwruc', 'sfwrun', 'sfwcrn', 'sfwcru', 'sfwcnr', 'sfwcnu', 'sfwcur', 'sfwcun', 'sfwnrc', 'sfwnru', 'sfwncr', 'sfwncu', 'sfwnur', 'sfwnuc', 'sfwurc', 'sfwurn', 'sfwucr', 'sfwucn', 'sfwunr', 'sfwunc', 'sfcrwn', 'sfcrwu', 'sfcrnw', 'sfcrnu', 'sfcruw', 'sfcrun', 'sfcwrn', 'sfcwru', 'sfcwnr', 'sfcwnu', 'sfcwur', 'sfcwun', 'sfcnrw', 'sfcnru', 'sfcnwr', 'sfcnwu', 'sfcnur', 'sfcnuw', 'sfcurw', 'sfcurn', 'sfcuwr', 'sfcuwn', 'sfcunr', 'sfcunw', 'sfnrwc', 'sfnrwu', 'sfnrcw', 'sfnrcu', 'sfnruw', 'sfnruc', 'sfnwrc', 'sfnwru', 'sfnwcr', 'sfnwcu', 'sfnwur', 'sfnwuc', 'sfncrw', 'sfncru', 'sfncwr', 'sfncwu', 'sfncur', 'sfncuw', 'sfnurw', 'sfnurc', 'sfnuwr', 'sfnuwc', 'sfnucr', 'sfnucw', 'sfurwc', 'sfurwn', 'sfurcw', 'sfurcn', 'sfurnw', 'sfurnc', 'sfuwrc', 'sfuwrn', 'sfuwcr', 'sfuwcn', 'sfuwnr', 'sfuwnc', 'sfucrw', 'sfucrn', 'sfucwr', 'sfucwn', 'sfucnr', 'sfucnw', 'sfunrw', 'sfunrc', 'sfunwr', 'sfunwc', 'sfuncr', 'sfuncw', 'surwcn', 'surwcf', 'surwnc', 'surwnf', 'surwfc', 'surwfn', 'surcwn', 'surcwf', 'surcnw', 'surcnf', 'surcfw', 'surcfn', 'surnwc', 'surnwf', 'surncw', 'surncf', 'surnfw', 'surnfc', 'surfwc', 'surfwn', 'surfcw', 'surfcn', 'surfnw', 'surfnc', 'suwrcn', 'suwrcf', 'suwrnc', 'suwrnf', 'suwrfc', 'suwrfn', 'suwcrn', 'suwcrf', 'suwcnr', 'suwcnf', 'suwcfr', 'suwcfn', 'suwnrc', 'suwnrf', 'suwncr', 'suwncf', 'suwnfr', 'suwnfc', 'suwfrc', 'suwfrn', 'suwfcr', 'suwfcn', 'suwfnr', 'suwfnc', 'sucrwn', 'sucrwf', 'sucrnw', 'sucrnf', 'sucrfw', 'sucrfn', 'sucwrn', 'sucwrf', 'sucwnr', 'sucwnf', 'sucwfr', 'sucwfn', 'sucnrw', 'sucnrf', 'sucnwr', 'sucnwf', 'sucnfr', 'sucnfw', 'sucfrw', 'sucfrn', 'sucfwr', 'sucfwn', 'sucfnr', 'sucfnw', 'sunrwc', 'sunrwf', 'sunrcw', 'sunrcf', 'sunrfw', 'sunrfc', 'sunwrc', 'sunwrf', 'sunwcr', 'sunwcf', 'sunwfr', 'sunwfc', 'suncrw', 'suncrf', 'suncwr', 'suncwf', 'suncfr', 'suncfw', 'sunfrw', 'sunfrc', 'sunfwr', 'sunfwc', 'sunfcr', 'sunfcw', 'sufrwc', 'sufrwn', 'sufrcw', 'sufrcn', 'sufrnw', 'sufrnc', 'sufwrc', 'sufwrn', 'sufwcr', 'sufwcn', 'sufwnr', 'sufwnc', 'sufcrw', 'sufcrn', 'sufcwr', 'sufcwn', 'sufcnr', 'sufcnw', 'sufnrw', 'sufnrc', 'sufnwr', 'sufnwc', 'sufncr', 'sufncw', 'urwcnf', 'urwcns', 'urwcfn', 'urwcfs', 'urwcsn', 'urwcsf', 'urwncf', 'urwncs', 'urwnfc', 'urwnfs', 'urwnsc', 'urwnsf', 'urwfcn', 'urwfcs', 'urwfnc', 'urwfns', 'urwfsc', 'urwfsn', 'urwscn', 'urwscf', 'urwsnc', 'urwsnf', 'urwsfc', 'urwsfn', 'urcwnf', 'urcwns', 'urcwfn', 'urcwfs', 'urcwsn', 'urcwsf', 'urcnwf', 'urcnws', 'urcnfw', 'urcnfs', 'urcnsw', 'urcnsf', 'urcfwn', 'urcfws', 'urcfnw', 'urcfns', 'urcfsw', 'urcfsn', 'urcswn', 'urcswf', 'urcsnw', 'urcsnf', 'urcsfw', 'urcsfn', 'urnwcf', 'urnwcs', 'urnwfc', 'urnwfs', 'urnwsc', 'urnwsf', 'urncwf', 'urncws', 'urncfw', 'urncfs', 'urncsw', 'urncsf', 'urnfwc', 'urnfws', 'urnfcw', 'urnfcs', 'urnfsw', 'urnfsc', 'urnswc', 'urnswf', 'urnscw', 'urnscf', 'urnsfw', 'urnsfc', 'urfwcn', 'urfwcs', 'urfwnc', 'urfwns', 'urfwsc', 'urfwsn', 'urfcwn', 'urfcws', 'urfcnw', 'urfcns', 'urfcsw', 'urfcsn', 'urfnwc', 'urfnws', 'urfncw', 'urfncs', 'urfnsw', 'urfnsc', 'urfswc', 'urfswn', 'urfscw', 'urfscn', 'urfsnw', 'urfsnc', 'urswcn', 'urswcf', 'urswnc', 'urswnf', 'urswfc', 'urswfn', 'urscwn', 'urscwf', 'urscnw', 'urscnf', 'urscfw', 'urscfn', 'ursnwc', 'ursnwf', 'ursncw', 'ursncf', 'ursnfw', 'ursnfc', 'ursfwc', 'ursfwn', 'ursfcw', 'ursfcn', 'ursfnw', 'ursfnc', 'uwrcnf', 'uwrcns', 'uwrcfn', 'uwrcfs', 'uwrcsn', 'uwrcsf', 'uwrncf', 'uwrncs', 'uwrnfc', 'uwrnfs', 'uwrnsc', 'uwrnsf', 'uwrfcn', 'uwrfcs', 'uwrfnc', 'uwrfns', 'uwrfsc', 'uwrfsn', 'uwrscn', 'uwrscf', 'uwrsnc', 'uwrsnf', 'uwrsfc', 'uwrsfn', 'uwcrnf', 'uwcrns', 'uwcrfn', 'uwcrfs', 'uwcrsn', 'uwcrsf', 'uwcnrf', 'uwcnrs', 'uwcnfr', 'uwcnfs', 'uwcnsr', 'uwcnsf', 'uwcfrn', 'uwcfrs', 'uwcfnr', 'uwcfns', 'uwcfsr', 'uwcfsn', 'uwcsrn', 'uwcsrf', 'uwcsnr', 'uwcsnf', 'uwcsfr', 'uwcsfn', 'uwnrcf', 'uwnrcs', 'uwnrfc', 'uwnrfs', 'uwnrsc', 'uwnrsf', 'uwncrf', 'uwncrs', 'uwncfr', 'uwncfs', 'uwncsr', 'uwncsf', 'uwnfrc', 'uwnfrs', 'uwnfcr', 'uwnfcs', 'uwnfsr', 'uwnfsc', 'uwnsrc', 'uwnsrf', 'uwnscr', 'uwnscf', 'uwnsfr', 'uwnsfc', 'uwfrcn', 'uwfrcs', 'uwfrnc', 'uwfrns', 'uwfrsc', 'uwfrsn', 'uwfcrn', 'uwfcrs', 'uwfcnr', 'uwfcns', 'uwfcsr', 'uwfcsn', 'uwfnrc', 'uwfnrs', 'uwfncr', 'uwfncs', 'uwfnsr', 'uwfnsc', 'uwfsrc', 'uwfsrn', 'uwfscr', 'uwfscn', 'uwfsnr', 'uwfsnc', 'uwsrcn', 'uwsrcf', 'uwsrnc', 'uwsrnf', 'uwsrfc', 'uwsrfn', 'uwscrn', 'uwscrf', 'uwscnr', 'uwscnf', 'uwscfr', 'uwscfn', 'uwsnrc', 'uwsnrf', 'uwsncr', 'uwsncf', 'uwsnfr', 'uwsnfc', 'uwsfrc', 'uwsfrn', 'uwsfcr', 'uwsfcn', 'uwsfnr', 'uwsfnc', 'ucrwnf', 'ucrwns', 'ucrwfn', 'ucrwfs', 'ucrwsn', 'ucrwsf', 'ucrnwf', 'ucrnws', 'ucrnfw', 'ucrnfs', 'ucrnsw', 'ucrnsf', 'ucrfwn', 'ucrfws', 'ucrfnw', 'ucrfns', 'ucrfsw', 'ucrfsn', 'ucrswn', 'ucrswf', 'ucrsnw', 'ucrsnf', 'ucrsfw', 'ucrsfn', 'ucwrnf', 'ucwrns', 'ucwrfn', 'ucwrfs', 'ucwrsn', 'ucwrsf', 'ucwnrf', 'ucwnrs', 'ucwnfr', 'ucwnfs', 'ucwnsr', 'ucwnsf', 'ucwfrn', 'ucwfrs', 'ucwfnr', 'ucwfns', 'ucwfsr', 'ucwfsn', 'ucwsrn', 'ucwsrf', 'ucwsnr', 'ucwsnf', 'ucwsfr', 'ucwsfn', 'ucnrwf', 'ucnrws', 'ucnrfw', 'ucnrfs', 'ucnrsw', 'ucnrsf', 'ucnwrf', 'ucnwrs', 'ucnwfr', 'ucnwfs', 'ucnwsr', 'ucnwsf', 'ucnfrw', 'ucnfrs', 'ucnfwr', 'ucnfws', 'ucnfsr', 'ucnfsw', 'ucnsrw', 'ucnsrf', 'ucnswr', 'ucnswf', 'ucnsfr', 'ucnsfw', 'ucfrwn', 'ucfrws', 'ucfrnw', 'ucfrns', 'ucfrsw', 'ucfrsn', 'ucfwrn', 'ucfwrs', 'ucfwnr', 'ucfwns', 'ucfwsr', 'ucfwsn', 'ucfnrw', 'ucfnrs', 'ucfnwr', 'ucfnws', 'ucfnsr', 'ucfnsw', 'ucfsrw', 'ucfsrn', 'ucfswr', 'ucfswn', 'ucfsnr', 'ucfsnw', 'ucsrwn', 'ucsrwf', 'ucsrnw', 'ucsrnf', 'ucsrfw', 'ucsrfn', 'ucswrn', 'ucswrf', 'ucswnr', 'ucswnf', 'ucswfr', 'ucswfn', 'ucsnrw', 'ucsnrf', 'ucsnwr', 'ucsnwf', 'ucsnfr', 'ucsnfw', 'ucsfrw', 'ucsfrn', 'ucsfwr', 'ucsfwn', 'ucsfnr', 'ucsfnw', 'unrwcf', 'unrwcs', 'unrwfc', 'unrwfs', 'unrwsc', 'unrwsf', 'unrcwf', 'unrcws', 'unrcfw', 'unrcfs', 'unrcsw', 'unrcsf', 'unrfwc', 'unrfws', 'unrfcw', 'unrfcs', 'unrfsw', 'unrfsc', 'unrswc', 'unrswf', 'unrscw', 'unrscf', 'unrsfw', 'unrsfc', 'unwrcf', 'unwrcs', 'unwrfc', 'unwrfs', 'unwrsc', 'unwrsf', 'unwcrf', 'unwcrs', 'unwcfr', 'unwcfs', 'unwcsr', 'unwcsf', 'unwfrc', 'unwfrs', 'unwfcr', 'unwfcs', 'unwfsr', 'unwfsc', 'unwsrc', 'unwsrf', 'unwscr', 'unwscf', 'unwsfr', 'unwsfc', 'uncrwf', 'uncrws', 'uncrfw', 'uncrfs', 'uncrsw', 'uncrsf', 'uncwrf', 'uncwrs', 'uncwfr', 'uncwfs', 'uncwsr', 'uncwsf', 'uncfrw', 'uncfrs', 'uncfwr', 'uncfws', 'uncfsr', 'uncfsw', 'uncsrw', 'uncsrf', 'uncswr', 'uncswf', 'uncsfr', 'uncsfw', 'unfrwc', 'unfrws', 'unfrcw', 'unfrcs', 'unfrsw', 'unfrsc', 'unfwrc', 'unfwrs', 'unfwcr', 'unfwcs', 'unfwsr', 'unfwsc', 'unfcrw', 'unfcrs', 'unfcwr', 'unfcws', 'unfcsr', 'unfcsw', 'unfsrw', 'unfsrc', 'unfswr', 'unfswc', 'unfscr', 'unfscw', 'unsrwc', 'unsrwf', 'unsrcw', 'unsrcf', 'unsrfw', 'unsrfc', 'unswrc', 'unswrf', 'unswcr', 'unswcf', 'unswfr', 'unswfc', 'unscrw', 'unscrf', 'unscwr', 'unscwf', 'unscfr', 'unscfw', 'unsfrw', 'unsfrc', 'unsfwr', 'unsfwc', 'unsfcr', 'unsfcw', 'ufrwcn', 'ufrwcs', 'ufrwnc', 'ufrwns', 'ufrwsc', 'ufrwsn', 'ufrcwn', 'ufrcws', 'ufrcnw', 'ufrcns', 'ufrcsw', 'ufrcsn', 'ufrnwc', 'ufrnws', 'ufrncw', 'ufrncs', 'ufrnsw', 'ufrnsc', 'ufrswc', 'ufrswn', 'ufrscw', 'ufrscn', 'ufrsnw', 'ufrsnc', 'ufwrcn', 'ufwrcs', 'ufwrnc', 'ufwrns', 'ufwrsc', 'ufwrsn', 'ufwcrn', 'ufwcrs', 'ufwcnr', 'ufwcns', 'ufwcsr', 'ufwcsn', 'ufwnrc', 'ufwnrs', 'ufwncr', 'ufwncs', 'ufwnsr', 'ufwnsc', 'ufwsrc', 'ufwsrn', 'ufwscr', 'ufwscn', 'ufwsnr', 'ufwsnc', 'ufcrwn', 'ufcrws', 'ufcrnw', 'ufcrns', 'ufcrsw', 'ufcrsn', 'ufcwrn', 'ufcwrs', 'ufcwnr', 'ufcwns', 'ufcwsr', 'ufcwsn', 'ufcnrw', 'ufcnrs', 'ufcnwr', 'ufcnws', 'ufcnsr', 'ufcnsw', 'ufcsrw', 'ufcsrn', 'ufcswr', 'ufcswn', 'ufcsnr', 'ufcsnw', 'ufnrwc', 'ufnrws', 'ufnrcw', 'ufnrcs', 'ufnrsw', 'ufnrsc', 'ufnwrc', 'ufnwrs', 'ufnwcr', 'ufnwcs', 'ufnwsr', 'ufnwsc', 'ufncrw', 'ufncrs', 'ufncwr', 'ufncws', 'ufncsr', 'ufncsw', 'ufnsrw', 'ufnsrc', 'ufnswr', 'ufnswc', 'ufnscr', 'ufnscw', 'ufsrwc', 'ufsrwn', 'ufsrcw', 'ufsrcn', 'ufsrnw', 'ufsrnc', 'ufswrc', 'ufswrn', 'ufswcr', 'ufswcn', 'ufswnr', 'ufswnc', 'ufscrw', 'ufscrn', 'ufscwr', 'ufscwn', 'ufscnr', 'ufscnw', 'ufsnrw', 'ufsnrc', 'ufsnwr', 'ufsnwc', 'ufsncr', 'ufsncw', 'usrwcn', 'usrwcf', 'usrwnc', 'usrwnf', 'usrwfc', 'usrwfn', 'usrcwn', 'usrcwf', 'usrcnw', 'usrcnf', 'usrcfw', 'usrcfn', 'usrnwc', 'usrnwf', 'usrncw', 'usrncf', 'usrnfw', 'usrnfc', 'usrfwc', 'usrfwn', 'usrfcw', 'usrfcn', 'usrfnw', 'usrfnc', 'uswrcn', 'uswrcf', 'uswrnc', 'uswrnf', 'uswrfc', 'uswrfn', 'uswcrn', 'uswcrf', 'uswcnr', 'uswcnf', 'uswcfr', 'uswcfn', 'uswnrc', 'uswnrf', 'uswncr', 'uswncf', 'uswnfr', 'uswnfc', 'uswfrc', 'uswfrn', 'uswfcr', 'uswfcn', 'uswfnr', 'uswfnc', 'uscrwn', 'uscrwf', 'uscrnw', 'uscrnf', 'uscrfw', 'uscrfn', 'uscwrn', 'uscwrf', 'uscwnr', 'uscwnf', 'uscwfr', 'uscwfn', 'uscnrw', 'uscnrf', 'uscnwr', 'uscnwf', 'uscnfr', 'uscnfw', 'uscfrw', 'uscfrn', 'uscfwr', 'uscfwn', 'uscfnr', 'uscfnw', 'usnrwc', 'usnrwf', 'usnrcw', 'usnrcf', 'usnrfw', 'usnrfc', 'usnwrc', 'usnwrf', 'usnwcr', 'usnwcf', 'usnwfr', 'usnwfc', 'usncrw', 'usncrf', 'usncwr', 'usncwf', 'usncfr', 'usncfw', 'usnfrw', 'usnfrc', 'usnfwr', 'usnfwc', 'usnfcr', 'usnfcw', 'usfrwc', 'usfrwn', 'usfrcw', 'usfrcn', 'usfrnw', 'usfrnc', 'usfwrc', 'usfwrn', 'usfwcr', 'usfwcn', 'usfwnr', 'usfwnc', 'usfcrw', 'usfcrn', 'usfcwr', 'usfcwn', 'usfcnr', 'usfcnw', 'usfnrw', 'usfnrc', 'usfnwr', 'usfnwc', 'usfncr', 'usfncw']
```

so flags could be one of these, right?

Reply:
I have managed to hack my brain!

Reply:
i think permutations are also allowed like "cs" and "sc"

Reply:
No, because the first character can only be `rwcn`. This is correct I believe: `[head + "".join(tail) for tail in [perm for i in range(4) for perm in permutations("fsu", i)] for head in "rwcn"]`.

Reply:
Yup!

Reply:
Sorry. I was AKF for some reason. I'll commit these changes.